[INFO] Starting execution.
[INFO] Downloading package list from PyPI ...
[INFO] (v) This package is up to date.
[INFO] (v-bootstrap) Data has been updated for this package.
[INFO] (v-crypt) This package is up to date.
[INFO] (v-ft) Data has been updated for this package.
[INFO] (v-histpy) This package is up to date.
[INFO] (v-learn-nester) This package is up to date.
[INFO] (v-log) Data has been updated for this package.
[INFO] (v-nester) This package is up to date.
[INFO] (v-one-sl) This package is up to date.
[INFO] (v-palette) This package is up to date.
[INFO] (v-time) Data has been updated for this package.
[INFO] (v-tools) Data has been updated for this package.
[INFO] (v-vk-api) This package is up to date.
[INFO] (v12) This package is up to date.
[INFO] (v1pysdk) This package is up to date.
[INFO] (v1pysdk-unofficial) This package is up to date.
[INFO] (v1syncrt) This package is up to date.
[INFO] (v2) This package is up to date.
[INFO] (v2-theme) This package is up to date.
[INFO] (v20) This package is up to date.
[INFO] (v2cjson) This package is up to date.
[INFO] (v2d) This package is up to date.
[INFO] (v2ex-daily-mission) This package is up to date.
[INFO] (v2extool) This package is up to date.
[ERROR] (v2f) This package does not have downloadable releases.
[INFO] (v2ray-auto-config) Data has been updated for this package.
[INFO] (v2ray-ping) This package is up to date.
[INFO] (v2ray-stats) Data has been updated for this package.
[INFO] (v2ray-util) Data has been updated for this package.
[INFO] (v2rayr) Data has been updated for this package.
[INFO] (v2rayu) Data has been updated for this package.
[ERROR] (v2sub) Could not extract data from this package. See below for details.
Unsupported package directory structure.
[INFO] (v3io) This package is up to date.
[INFO] (v3io-frames) Data has been updated for this package.
[INFO] (v3n0m) This package is up to date.
[INFO] (v4) This package is up to date.
[INFO] (v4docker) Data has been updated for this package.
[INFO] (v4l2) This package is up to date.
[INFO] (v4l2capture) This package is up to date.
[ERROR] (v4py) This package does not have downloadable releases.
[INFO] (v7py) This package is up to date.
[INFO] (v7xstyle) Data has been updated for this package.
[INFO] (v8) This package is up to date.
[INFO] (v8-cffi) This package is up to date.
[INFO] (v8bot) This package is up to date.
[INFO] (v8eval) This package is up to date.
[INFO] (v8py) This package is up to date.
[INFO] (v9) This package is up to date.
[INFO] (va) Data has been updated for this package.
[INFO] (va-samba-api) This package is up to date.
[INFO] (vaa) Data has been updated for this package.
[ERROR] (vaalikone) This package does not have downloadable releases.
[INFO] (vabbat) Data has been updated for this package.
[INFO] (vac) This package is up to date.
[INFO] (vac-exporter) This package is up to date.
[INFO] (vac-templater) This package is up to date.
[INFO] (vac2fost) This package is up to date.
[INFO] (vacances-scolaires-france) Data has been updated for this package.
[INFO] (vacasa) This package is up to date.
[INFO] (vacation) This package is up to date.
[INFO] (vacca) This package is up to date.
[INFO] (vaccine) This package is up to date.
[INFO] (vacumm) Data has been updated for this package.
[INFO] (vacumm-data) This package is up to date.
[INFO] (vacuum) This package is up to date.
[INFO] (vacuum-cleaner) This package is up to date.
[INFO] (vacuumworld) Data has been updated for this package.
[ERROR] (vaddipar) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vader-multi) Data has been updated for this package.
[INFO] (vader-sentiment) This package is up to date.
[INFO] (vadersentiment) This package is up to date.
[INFO] (vadersentiment-swedish) Data has been updated for this package.
[INFO] (vadersentimentcustom) This package is up to date.
[INFO] (vadjpdf) This package is up to date.
[INFO] (vadm) This package is up to date.
[INFO] (vadtk) This package is up to date.
[INFO] (vae) Data has been updated for this package.
[INFO] (vae-oversampler) Data has been updated for this package.
[INFO] (vae1) Data has been updated for this package.
[INFO] (vae2) Data has been updated for this package.
[INFO] (vaex) Data has been updated for this package.
[INFO] (vaex-arrow) Data has been updated for this package.
[INFO] (vaex-astro) Data has been updated for this package.
[INFO] (vaex-core) Data has been updated for this package.
[INFO] (vaex-distributed) This package is up to date.
[INFO] (vaex-graphql) Data has been updated for this package.
[INFO] (vaex-hdf5) Data has been updated for this package.
[INFO] (vaex-jupyter) Data has been updated for this package.
[INFO] (vaex-ml) Data has been updated for this package.
[INFO] (vaex-server) This package is up to date.
[INFO] (vaex-ui) Data has been updated for this package.
[INFO] (vaex-viz) Data has been updated for this package.
[INFO] (vagabond) This package is up to date.
[INFO] (vagalume) This package is up to date.
[ERROR] (vagoth) This package does not have downloadable releases.
[INFO] (vagquery) This package is up to date.
[INFO] (vagrant-environment) This package is up to date.
[INFO] (vagrant-metadata) This package is up to date.
[INFO] (vagrant-playbook) This package is up to date.
[INFO] (vagrant2json) This package is up to date.
[INFO] (vagrantor) This package is up to date.
[INFO] (vagrantpy) Data has been updated for this package.
[INFO] (vagranttoansible) This package is up to date.
[INFO] (vagrepo) This package is up to date.
[INFO] (vai) This package is up to date.
[INFO] (vais) This package is up to date.
[INFO] (vais-plug) Data has been updated for this package.
[INFO] (vaisasr-python) Data has been updated for this package.
[INFO] (vaisdemo) This package is up to date.
[INFO] (vaishrg) Data has been updated for this package.
[INFO] (vaiskit) This package is up to date.
[INFO] (vaitk) This package is up to date.
[INFO] (vak) Data has been updated for this package.
[INFO] (vak-test-net) This package is up to date.
[INFO] (vakdocker) This package is up to date.
[ERROR] (vake) This package does not have downloadable releases.
[INFO] (vakhshour) This package is up to date.
[INFO] (vakkenranking) This package is up to date.
[INFO] (vaksana-proto) This package is up to date.
[INFO] (vaksana-sdk) This package is up to date.
[INFO] (vakt) Data has been updated for this package.
[INFO] (vaktuk) This package is up to date.
[INFO] (vakuum) This package is up to date.
[INFO] (val) This package is up to date.
[INFO] (valarmath) This package is up to date.
[INFO] (valbot) Data has been updated for this package.
[INFO] (valclust) This package is up to date.
[INFO] (valection) This package is up to date.
[INFO] (valedictory) Data has been updated for this package.
[INFO] (valeera) This package is up to date.
[INFO] (valence) This package is up to date.
[INFO] (valentine-contentportlets) This package is up to date.
[INFO] (valentine-imagescales) This package is up to date.
[INFO] (valentine-languagequery) This package is up to date.
[INFO] (valentine-linguaflow) This package is up to date.
[INFO] (valentine-multiparagraphfield) This package is up to date.
[INFO] (valentine-multiparagraphpage) This package is up to date.
[INFO] (valerius) This package is up to date.
[INFO] (valery) This package is up to date.
[INFO] (valet) This package is up to date.
[ERROR] (valetion) This package does not have downloadable releases.
[INFO] (valgreen) Data has been updated for this package.
[INFO] (valgrind-parser) Data has been updated for this package.
[ERROR] (valhal) This package does not have downloadable releases.
[INFO] (valhalla) This package is up to date.
[INFO] (valhallaapi) This package is up to date.
[INFO] (valhallapy) This package is up to date.
[INFO] (vali) This package is up to date.
[INFO] (valid) This package is up to date.
[INFO] (valid-edtf) This package is up to date.
[INFO] (valid-email) This package is up to date.
[INFO] (valid-model) This package is up to date.
[INFO] (valid-pymail) This package is up to date.
[INFO] (valid8) Data has been updated for this package.
[INFO] (validada) This package is up to date.
[INFO] (validata) This package is up to date.
[INFO] (validata-api) Data has been updated for this package.
[INFO] (validata-core) Data has been updated for this package.
[INFO] (validata-ui) Data has been updated for this package.
[INFO] (validate) This package is up to date.
[INFO] (validate-aws-sns-message) Data has been updated for this package.
[INFO] (validate-docbr) Data has been updated for this package.
[INFO] (validate-email) This package is up to date.
[INFO] (validate-email-address) This package is up to date.
[INFO] (validate-email-wt) This package is up to date.
[INFO] (validate-it) Data has been updated for this package.
[INFO] (validate-nigerian-phone) Data has been updated for this package.
[INFO] (validate-parameters) This package is up to date.
[INFO] (validate-utf8) This package is up to date.
[INFO] (validate-version-code) This package is up to date.
[INFO] (validate-yaml) This package is up to date.
[INFO] (validate-yml) This package is up to date.
[INFO] (validatecodetool) This package is up to date.
[ERROR] (validated) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (validatedoc) This package is up to date.
[INFO] (validateemail) This package is up to date.
[INFO] (validateit) This package is up to date.
[INFO] (validater) This package is up to date.
[INFO] (validatesns) This package is up to date.
[INFO] (validatexmls) This package is up to date.
[INFO] (validation) This package is up to date.
[INFO] (validation-app-engine) Data has been updated for this package.
[INFO] (validation-coding) This package is up to date.
[INFO] (validation-decorators) This package is up to date.
[INFO] (validation-utils) Data has been updated for this package.
[INFO] (validation21) This package is up to date.
[ERROR] (validationlib) This package does not have downloadable releases.
[INFO] (validations) This package is up to date.
[INFO] (validationtool) Data has been updated for this package.
[INFO] (validatish) This package is up to date.
[INFO] (validator) This package is up to date.
[INFO] (validator-collection) Data has been updated for this package.
[INFO] (validator-collection-br) Data has been updated for this package.
[INFO] (validator-decorator) Data has been updated for this package.
[INFO] (validator-devel) This package is up to date.
[INFO] (validator-helper) Data has been updated for this package.
[ERROR] (validator-pkg) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (validator-py) This package is up to date.
[INFO] (validatorpy) This package is up to date.
[INFO] (validators) Data has been updated for this package.
[INFO] (validclust) This package is up to date.
[INFO] (valideer) This package is up to date.
[INFO] (validemail) This package is up to date.
[INFO] (validfilter) This package is up to date.
[INFO] (validframes) Data has been updated for this package.
[INFO] (validict) This package is up to date.
[INFO] (validictory) This package is up to date.
[INFO] (validino) This package is up to date.
[INFO] (validity) This package is up to date.
[INFO] (validjson) This package is up to date.
[INFO] (validobj) Data has been updated for this package.
[INFO] (validol) This package is up to date.
[INFO] (validol2) This package is up to date.
[INFO] (validoot) This package is up to date.
[ERROR] (validpie) This package does not have downloadable releases.
[INFO] (validr) Data has been updated for this package.
[INFO] (validus) This package is up to date.
[INFO] (validus-pkg-test) This package is up to date.
[INFO] (validx) This package is up to date.
[INFO] (validyaml) This package is up to date.
[INFO] (valifor) This package is up to date.
[INFO] (valigator) This package is up to date.
[INFO] (valiml) Data has been updated for this package.
[INFO] (valinor) This package is up to date.
[INFO] (valiot-worker) This package is up to date.
[INFO] (valiotworker) Data has been updated for this package.
[INFO] (valipede) This package is up to date.
[INFO] (valis) This package is up to date.
[INFO] (valispace) Data has been updated for this package.
[INFO] (valkit) This package is up to date.
[INFO] (valkka-live) This package is up to date.
[INFO] (valkyrie) Data has been updated for this package.
[INFO] (valleorm) This package is up to date.
[INFO] (valley) This package is up to date.
[INFO] (valleybackups) This package is up to date.
[INFO] (valleydata) This package is up to date.
[INFO] (valleydeight) This package is up to date.
[INFO] (vallox-websocket-api) Data has been updated for this package.
[INFO] (valohai-cli) Data has been updated for this package.
[INFO] (valohai-local-run) This package is up to date.
[INFO] (valohai-yaml) Data has been updated for this package.
[INFO] (valor) This package is up to date.
[INFO] (valorum) This package is up to date.
[INFO] (valpy) This package is up to date.
[INFO] (valr-python) Data has been updated for this package.
[INFO] (vals) This package is up to date.
[INFO] (valstream) This package is up to date.
[INFO] (valtioneuvosto-scraper) This package is up to date.
[INFO] (valuable) This package is up to date.
[ERROR] (valuationframework) This package does not have downloadable releases.
[INFO] (value) This package is up to date.
[INFO] (value-object) This package is up to date.
[INFO] (value-objects) This package is up to date.
[INFO] (valuedispatch) This package is up to date.
[INFO] (valuehorizon-companies) This package is up to date.
[INFO] (valuehorizon-countries) This package is up to date.
[INFO] (valuehorizon-forex) This package is up to date.
[INFO] (valuehorizon-people) This package is up to date.
[INFO] (valueobject) This package is up to date.
[INFO] (values) This package is up to date.
[INFO] (valve) This package is up to date.
[INFO] (valve-monitor) This package is up to date.
[INFO] (valve-range-query) This package is up to date.
[INFO] (vam) This package is up to date.
[INFO] (vam-whittaker) Data has been updated for this package.
[INFO] (vame-ir) This package is up to date.
[INFO] (vamp) This package is up to date.
[INFO] (vampire) This package is up to date.
[INFO] (vampireanalysis) Data has been updated for this package.
[INFO] (vampireformac) This package is up to date.
[INFO] (vampireingalaxy) This package is up to date.
[ERROR] (vampiremtest) This package does not have downloadable releases.
[INFO] (van) This package is up to date.
[INFO] (van-api) This package is up to date.
[INFO] (van-contactology) This package is up to date.
[INFO] (van-pg) This package is up to date.
[INFO] (van-potomo) This package is up to date.
[INFO] (van-pydeb) This package is up to date.
[INFO] (van-reposync) This package is up to date.
[INFO] (van-resumelb) This package is up to date.
[INFO] (van-static) This package is up to date.
[INFO] (van-testing) This package is up to date.
[INFO] (van-timeformat) This package is up to date.
[INFO] (vanadis) Data has been updated for this package.
[INFO] (vanadium) This package is up to date.
[INFO] (vancare-fact) This package is up to date.
[INFO] (vanda) This package is up to date.
[INFO] (vander) This package is up to date.
[INFO] (vandermonde) This package is up to date.
[INFO] (vandroguard) This package is up to date.
[INFO] (vane) This package is up to date.
[INFO] (vane2) Data has been updated for this package.
[INFO] (vanellope) This package is up to date.
[INFO] (vanescosc2) This package is up to date.
[INFO] (vaneska) This package is up to date.
[INFO] (vang) This package is up to date.
[INFO] (vanguard) This package is up to date.
[INFO] (vanguardistas-buildoutsvn) This package is up to date.
[INFO] (vanguardistas-pydebdep) This package is up to date.
[INFO] (vanguardkit) Data has been updated for this package.
[INFO] (vanguards) This package is up to date.
[INFO] (vanhozone) This package is up to date.
[INFO] (vania) This package is up to date.
[INFO] (vanilla) This package is up to date.
[INFO] (vanilla-bean) This package is up to date.
[INFO] (vanilla-consul) This package is up to date.
[INFO] (vanilla-option-pricing) This package is up to date.
[INFO] (vanilla-pkg) This package is up to date.
[INFO] (vanity) This package is up to date.
[INFO] (vanity-address) This package is up to date.
[INFO] (vanitynumber) Data has been updated for this package.
[INFO] (vanityonionr) Data has been updated for this package.
[INFO] (vannistell) This package is up to date.
[INFO] (vansel0104) This package is up to date.
[INFO] (vantage) Data has been updated for this package.
[INFO] (vantetider-scraper) This package is up to date.
[INFO] (vantivecommercesdk) Data has been updated for this package.
[INFO] (vape) This package is up to date.
[INFO] (vapeplot) This package is up to date.
[INFO] (vapetool) Data has been updated for this package.
[INFO] (vaping) Data has been updated for this package.
[INFO] (vaporize) This package is up to date.
[INFO] (vaporwavely) This package is up to date.
[INFO] (vapory) This package is up to date.
[INFO] (vapory-git) This package is up to date.
[INFO] (vapour-linux-amd64) This package is up to date.
[INFO] (vapour-windows-amd64) This package is up to date.
[INFO] (vapourapps) This package is up to date.
[INFO] (vapoursynth) Data has been updated for this package.
[INFO] (vapoursynth-portable) Data has been updated for this package.
[INFO] (vapprun) This package is up to date.
[INFO] (vapps) This package is up to date.
[INFO] (vapr) This package is up to date.
[INFO] (vaptcha) This package is up to date.
[INFO] (vaptchasdk) This package is up to date.
[INFO] (vapy) Data has been updated for this package.
[INFO] (vaquero) This package is up to date.
[INFO] (vaquita) This package is up to date.
[INFO] (var) This package is up to date.
[INFO] (var-control) This package is up to date.
[INFO] (var-dump) This package is up to date.
[ERROR] (var-dumpy) This package does not have downloadable releases.
[INFO] (varaha) Data has been updated for this package.
[INFO] (varakumar1105-add-numbers) This package is up to date.
[INFO] (varana) This package is up to date.
[INFO] (varapp-backend-py) This package is up to date.
[INFO] (varas) This package is up to date.
[INFO] (varbio) This package is up to date.
[INFO] (varblez1-stage-a) This package is up to date.
[INFO] (varclushi) This package is up to date.
[ERROR] (varclushi-jingtt) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (varcode) Data has been updated for this package.
[INFO] (varda) This package is up to date.
[INFO] (vardbg) Data has been updated for this package.
[INFO] (varenv) Data has been updated for this package.
[INFO] (varevents) This package is up to date.
[INFO] (varex) This package is up to date.
[INFO] (varg) Data has been updated for this package.
[INFO] (vargas) This package is up to date.
[INFO] (vari) This package is up to date.
[INFO] (varia) This package is up to date.
[INFO] (variable) This package is up to date.
[INFO] (variable-generator) This package is up to date.
[INFO] (variable-updater) Data has been updated for this package.
[INFO] (variadic) This package is up to date.
[INFO] (variance) This package is up to date.
[INFO] (variance-finder) This package is up to date.
[INFO] (variant-integrity) This package is up to date.
[INFO] (variant-spark) Data has been updated for this package.
[INFO] (variant-tools) Data has been updated for this package.
[ERROR] (variant2pubmed) This package does not have downloadable releases.
[INFO] (variantannotation) This package is up to date.
[INFO] (variantgrid-api) This package is up to date.
[INFO] (variants) This package is up to date.
[INFO] (variations) Data has been updated for this package.
[INFO] (varibayes) This package is up to date.
[INFO] (varify) This package is up to date.
[INFO] (varify-client) This package is up to date.
[INFO] (varint) This package is up to date.
[ERROR] (varints) This package does not have downloadable releases.
[INFO] (variousdisplaywidgets) This package is up to date.
[INFO] (varlens) This package is up to date.
[INFO] (varlet) This package is up to date.
[INFO] (varlink) Data has been updated for this package.
[INFO] (varname) Data has been updated for this package.
[INFO] (varnish-admin-socket) This package is up to date.
[INFO] (varnish-bans-manager) This package is up to date.
[INFO] (varnishapi) This package is up to date.
[INFO] (varnishsentry) This package is up to date.
[INFO] (varoperhelper) This package is up to date.
[INFO] (varp) This package is up to date.
[INFO] (vars-pickle) Data has been updated for this package.
[INFO] (varseries) Data has been updated for this package.
[INFO] (varshith) This package is up to date.
[INFO] (varsity) This package is up to date.
[INFO] (varsnap) Data has been updated for this package.
[INFO] (varstar) This package is up to date.
[INFO] (varsvm) Data has been updated for this package.
[INFO] (vartrix) Data has been updated for this package.
[INFO] (varunmurthy) This package is up to date.
[INFO] (varyaml) This package is up to date.
[INFO] (varys) This package is up to date.
[INFO] (varz) Data has been updated for this package.
[INFO] (vas) This package is up to date.
[INFO] (vasalksyy-package) Data has been updated for this package.
[INFO] (vasalskyy) Data has been updated for this package.
[INFO] (vasctree) This package is up to date.
[INFO] (vase) This package is up to date.
[INFO] (vasp-tools) Data has been updated for this package.
[INFO] (vaspcz) Data has been updated for this package.
[INFO] (vasppy) This package is up to date.
[INFO] (vasprun) This package is up to date.
[INFO] (vasprun-xml) Data has been updated for this package.
[INFO] (vasptools) This package is up to date.
[INFO] (vaspwiki) This package is up to date.
[INFO] (vaspy) This package is up to date.
[INFO] (vassal) This package is up to date.
[INFO] (vassal-deployer) This package is up to date.
[ERROR] (vast) This package does not have downloadable releases.
[INFO] (vasuki) This package is up to date.
[INFO] (vat) This package is up to date.
[INFO] (vat-build-utils) Data has been updated for this package.
[INFO] (vat-format-checker) This package is up to date.
[INFO] (vat-moss) This package is up to date.
[INFO] (vat-package-utils) Data has been updated for this package.
[INFO] (vat-python-test-utils) This package is up to date.
[INFO] (vat-utils) Data has been updated for this package.
[INFO] (vat-validator) This package is up to date.
[INFO] (vatan) This package is up to date.
[INFO] (vater) Data has been updated for this package.
[INFO] (vatnumber) This package is up to date.
[INFO] (vatools) Data has been updated for this package.
[ERROR] (vatsag-nester) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vatsim-sso) This package is up to date.
[INFO] (vatstat) This package is up to date.
[INFO] (vaud) This package is up to date.
[INFO] (vault-anyconfig) Data has been updated for this package.
[INFO] (vault-ca) This package is up to date.
[INFO] (vault-certificate-deploy) Data has been updated for this package.
[INFO] (vault-cli) Data has been updated for this package.
[INFO] (vault-dev) This package is up to date.
[INFO] (vault-gatekeeper-client) This package is up to date.
[INFO] (vault-keepass-import) This package is up to date.
[INFO] (vault-pki-extractor) This package is up to date.
[INFO] (vault-printer) Data has been updated for this package.
[INFO] (vault-psycopg2) This package is up to date.
[INFO] (vault-redirector) This package is up to date.
[INFO] (vault-tools) Data has been updated for this package.
[INFO] (vault2env) Data has been updated for this package.
[INFO] (vaultcacli) This package is up to date.
[INFO] (vaultenv) This package is up to date.
[INFO] (vaultfly) This package is up to date.
[INFO] (vaultier) This package is up to date.
[INFO] (vaultkeeper) This package is up to date.
[INFO] (vaultkeeper-adaptor) This package is up to date.
[INFO] (vaultlib) This package is up to date.
[INFO] (vaultlocker) Data has been updated for this package.
[INFO] (vaulty) This package is up to date.
[INFO] (vaunix-api) This package is up to date.
[INFO] (vaurien) This package is up to date.
[INFO] (vaurienclient) This package is up to date.
[INFO] (vauseragent) This package is up to date.
[INFO] (vaw) This package is up to date.
[INFO] (vaxrank) This package is up to date.
[INFO] (vaydiff) This package is up to date.
[INFO] (vayu) Data has been updated for this package.
[ERROR] (vazaar) This package does not have downloadable releases.
[INFO] (vb-console) Data has been updated for this package.
[INFO] (vb-python) This package is up to date.
[INFO] (vb-toolbox) Data has been updated for this package.
[ERROR] (vb2py) Could not extract data from this package. See below for details.
".htm" extension not supported.
[INFO] (vba-wrapper) This package is up to date.
[ERROR] (vbae) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vbait) This package is up to date.
[INFO] (vbbinarylensing) This package is up to date.
[INFO] (vbbvg) This package is up to date.
[INFO] (vbcode) This package is up to date.
[INFO] (vbd) This package is up to date.
[INFO] (vbdiar) This package is up to date.
[ERROR] (vbdmount) Could not extract data from this package. See below for details.
".py" extension not supported.
[INFO] (vbeo-seantis-dir-events) This package is up to date.
[INFO] (vbftool) This package is up to date.
[INFO] (vbhelloworld) This package is up to date.
[INFO] (vbhv) This package is up to date.
[INFO] (vbihmm) This package is up to date.
[INFO] (vbindex) Data has been updated for this package.
[INFO] (vbio) This package is up to date.
[INFO] (vbiz-fetcher) Data has been updated for this package.
[INFO] (vbiz-parser) Data has been updated for this package.
[INFO] (vblapi) This package is up to date.
[INFO] (vbmfa) This package is up to date.
[INFO] (vbml) Data has been updated for this package.
[INFO] (vbox) This package is up to date.
[INFO] (vbox-cli) This package is up to date.
[INFO] (vbox-sdk) This package is up to date.
[ERROR] (vboxapi) This package does not have downloadable releases.
[ERROR] (vboxen) This package does not have downloadable releases.
[ERROR] (vboxn) Could not extract data from this package. See below for details.
"http://pypi.python.org/packages/source/v/vboxn/vboxn-0.2.4.tar.gz" file could not be downloaded. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 107, in download_tar
    f.write(urlopen(url=urlesc(pkgurl), timeout=10).read())
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: SSL is required

[INFO] (vboxoverlord) This package is up to date.
[INFO] (vboxtrayico) This package is up to date.
[INFO] (vbp) This package is up to date.
[INFO] (vbtools) This package is up to date.
[INFO] (vbuild) Data has been updated for this package.
[INFO] (vbuilder) Data has been updated for this package.
[INFO] (vbump) Data has been updated for this package.
[INFO] (vbx-tree) This package is up to date.
[INFO] (vbx-vk) This package is up to date.
[INFO] (vc) This package is up to date.
[INFO] (vc-deepmodel) This package is up to date.
[INFO] (vc-diss) This package is up to date.
[ERROR] (vc-diss-cruncher) This package does not have downloadable releases.
[ERROR] (vc0706serial) This package does not have downloadable releases.
[INFO] (vc2f1kjg2n) This package is up to date.
[ERROR] (vc2xlsx) This package does not have downloadable releases.
[INFO] (vc3-client) This package is up to date.
[INFO] (vc3-info-service) This package is up to date.
[INFO] (vca-cli) This package is up to date.
[INFO] (vcab) This package is up to date.
[INFO] (vcai-be-common) This package is up to date.
[ERROR] (vcalsax) This package does not have downloadable releases.
[INFO] (vcap) Data has been updated for this package.
[INFO] (vcap-services) This package is up to date.
[INFO] (vcap-utils) Data has been updated for this package.
[INFO] (vcapi) This package is up to date.
[INFO] (vcard) Data has been updated for this package.
[INFO] (vcard-module) This package is up to date.
[INFO] (vcard-split) This package is up to date.
[INFO] (vcardgen) This package is up to date.
[INFO] (vcardreader) This package is up to date.
[INFO] (vcardtools) This package is up to date.
[INFO] (vcardz-data) This package is up to date.
[INFO] (vcash-hash) This package is up to date.
[INFO] (vcc) This package is up to date.
[INFO] (vcd-cli) Data has been updated for this package.
[INFO] (vcdb) This package is up to date.
[INFO] (vcdextmessageworker) This package is up to date.
[INFO] (vcdextproxy) Data has been updated for this package.
[INFO] (vcdriver) This package is up to date.
[INFO] (vcdriver2) This package is up to date.
[INFO] (vcdvcd) This package is up to date.
[INFO] (vceffort) This package is up to date.
[INFO] (vcenter) This package is up to date.
[INFO] (vcenter-prometheus-exporter) This package is up to date.
[INFO] (vcert) Data has been updated for this package.
[INFO] (vcf-annotate-polyphen) This package is up to date.
[INFO] (vcf-annotation-tools) This package is up to date.
[INFO] (vcf-consensus-builder) Data has been updated for this package.
[INFO] (vcf-dedup) This package is up to date.
[INFO] (vcf-kit) This package is up to date.
[INFO] (vcf-parser) This package is up to date.
[INFO] (vcf2clinvar) This package is up to date.
[INFO] (vcf2cytosure) This package is up to date.
[INFO] (vcf2mutt) This package is up to date.
[INFO] (vcf2networks) This package is up to date.
[INFO] (vcfarray) This package is up to date.
[INFO] (vcfiterator) This package is up to date.
[INFO] (vcfminerclient) This package is up to date.
[INFO] (vcfnp) This package is up to date.
[INFO] (vcfparser) Data has been updated for this package.
[INFO] (vcfped) This package is up to date.
[INFO] (vcfplt) This package is up to date.
[INFO] (vcfproj) Data has been updated for this package.
[INFO] (vcfpy) This package is up to date.
[INFO] (vcfpy2) This package is up to date.
[INFO] (vcfremapper) Data has been updated for this package.
[INFO] (vcfstats) Data has been updated for this package.
[INFO] (vcfstoreclient) This package is up to date.
[INFO] (vcfsyncer) Data has been updated for this package.
[INFO] (vcftoabook) Data has been updated for this package.
[INFO] (vcftoolbox) This package is up to date.
[INFO] (vcftoolz) This package is up to date.
[INFO] (vcfviewer) This package is up to date.
[INFO] (vcfy) This package is up to date.
[INFO] (vcheck) This package is up to date.
[INFO] (vci) This package is up to date.
[INFO] (vckube) This package is up to date.
[INFO] (vcli) This package is up to date.
[INFO] (vcloudair) This package is up to date.
[INFO] (vcloudtools) This package is up to date.
[INFO] (vcoclient) Data has been updated for this package.
[INFO] (vcode-api) This package is up to date.
[INFO] (vcommand) This package is up to date.
[INFO] (vcomp) This package is up to date.
[INFO] (vcompress) This package is up to date.
[INFO] (vconfig) This package is up to date.
[INFO] (vconnect) This package is up to date.
[INFO] (vconnector) Data has been updated for this package.
[INFO] (vcontext) Data has been updated for this package.
[INFO] (vcontrol) This package is up to date.
[INFO] (vcopt) Data has been updated for this package.
[INFO] (vcore-api) This package is up to date.
[INFO] (vcosc) This package is up to date.
[INFO] (vcp) This package is up to date.
[ERROR] (vcprompt) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vcrpy) Data has been updated for this package.
[ERROR] (vcrpy-facebook) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vcrpy-unittest) This package is up to date.
[INFO] (vcrpyunittest) This package is up to date.
[INFO] (vcs) This package is up to date.
[INFO] (vcs-extras) This package is up to date.
[INFO] (vcs-latest) This package is up to date.
[INFO] (vcs-mirrors) This package is up to date.
[INFO] (vcs-repo-mgr) This package is up to date.
[INFO] (vcs-ssh) This package is up to date.
[INFO] (vcs-uploader) Data has been updated for this package.
[INFO] (vcs2eric) This package is up to date.
[INFO] (vcsdeploy) This package is up to date.
[INFO] (vcsecret) This package is up to date.
[INFO] (vcsi) Data has been updated for this package.
[INFO] (vcslinks) This package is up to date.
[INFO] (vcspull) This package is up to date.
[INFO] (vcsrosetta) This package is up to date.
[INFO] (vcsserver) This package is up to date.
[INFO] (vcsshark) Data has been updated for this package.
[ERROR] (vcssshd) This package does not have downloadable releases.
[INFO] (vcstodo) This package is up to date.
[INFO] (vcstool) Data has been updated for this package.
[INFO] (vcstools) Data has been updated for this package.
[INFO] (vcstools-latest) This package is up to date.
[INFO] (vctoolkit) Data has been updated for this package.
[INFO] (vctools) This package is up to date.
[INFO] (vcu-ui) Data has been updated for this package.
[INFO] (vcv) This package is up to date.
[INFO] (vcver) Data has been updated for this package.
[INFO] (vcversioner) This package is up to date.
[INFO] (vcvf) This package is up to date.
[INFO] (vcvf-emotion) This package is up to date.
[INFO] (vd42-openpyxl) This package is up to date.
[INFO] (vda) Data has been updated for this package.
[ERROR] (vdb) This package does not have downloadable releases.
[INFO] (vdbscan) This package is up to date.
[INFO] (vdd) Data has been updated for this package.
[INFO] (vdexcsv) This package is up to date.
[INFO] (vdf) Data has been updated for this package.
[INFO] (vdf2json) This package is up to date.
[INFO] (vdfedit) This package is up to date.
[INFO] (vdgalcon) This package is up to date.
[INFO] (vdhost) This package is up to date.
[ERROR] (vdi-generator) This package does not have downloadable releases.
[INFO] (vdian) This package is up to date.
[INFO] (vdiff) This package is up to date.
[INFO] (vdirect-client) Data has been updated for this package.
[INFO] (vdirsyncer) This package is up to date.
[INFO] (vdisk) This package is up to date.
[INFO] (vdist) Data has been updated for this package.
[INFO] (vdj) This package is up to date.
[INFO] (vdm) This package is up to date.
[INFO] (vdm3) This package is up to date.
[INFO] (vdms) This package is up to date.
[INFO] (vdom) This package is up to date.
[INFO] (vdomify) This package is up to date.
[INFO] (vdomr) This package is up to date.
[INFO] (vdq) Data has been updated for this package.
[INFO] (vdrnn) This package is up to date.
[INFO] (vdsgen) This package is up to date.
[INFO] (vdt-simpleaptrepo) This package is up to date.
[INFO] (vdt-version) This package is up to date.
[INFO] (vdt-versionplugin-buildout) This package is up to date.
[INFO] (vdt-versionplugin-debianize) This package is up to date.
[INFO] (vdt-versionplugin-default) This package is up to date.
[INFO] (vdt-versionplugin-fpm) This package is up to date.
[INFO] (vdt-versionplugin-gitchain) This package is up to date.
[INFO] (vdt-versionplugin-hotfix) This package is up to date.
[INFO] (vdt-versionplugin-puppetforge) This package is up to date.
[INFO] (vdt-versionplugin-puppetmodule) This package is up to date.
[INFO] (vdt-versionplugin-wheel) This package is up to date.
[INFO] (vdu-nlp-services) Data has been updated for this package.
[INFO] (vdupgrade) Data has been updated for this package.
[INFO] (vdv2pg) This package is up to date.
[INFO] (vdw) This package is up to date.
[INFO] (ve) Data has been updated for this package.
[INFO] (veb) This package is up to date.
[ERROR] (vebai-gists) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vebai-gists1) Data has been updated for this package.
[INFO] (vebits-api) Data has been updated for this package.
[INFO] (vebootstrap) This package is up to date.
[INFO] (vec-noise) This package is up to date.
[INFO] (vec2face) Data has been updated for this package.
[INFO] (vec2graph) This package is up to date.
[INFO] (vec2img) This package is up to date.
[INFO] (vecc) This package is up to date.
[INFO] (vecha) This package is up to date.
[INFO] (vechain) This package is up to date.
[INFO] (vecino) This package is up to date.
[INFO] (vecmaths) This package is up to date.
[INFO] (vecnet) This package is up to date.
[INFO] (vecnet-emod) This package is up to date.
[INFO] (vecnet-openmalaria) This package is up to date.
[INFO] (vecnet-simulation) This package is up to date.
[INFO] (vecnet-winhpc) This package is up to date.
[INFO] (vecpy) This package is up to date.
[INFO] (vecrec) This package is up to date.
[INFO] (vecshare) This package is up to date.
[INFO] (vecstack) Data has been updated for this package.
[INFO] (vecstatsgraph3d) This package is up to date.
[ERROR] (vectify) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vectionary) This package is up to date.
[INFO] (vecto) Data has been updated for this package.
[INFO] (vectools) This package is up to date.
[INFO] (vector) Data has been updated for this package.
[INFO] (vector-2d) This package is up to date.
[INFO] (vector-datasource) This package is up to date.
[INFO] (vector-demo) This package is up to date.
[INFO] (vector-shortcuts) Data has been updated for this package.
[INFO] (vector2d) This package is up to date.
[INFO] (vector3d) This package is up to date.
[INFO] (vectorcalculator) This package is up to date.
[INFO] (vectordash) This package is up to date.
[INFO] (vectordict) This package is up to date.
[INFO] (vectorfields) This package is up to date.
[INFO] (vectorformats) This package is up to date.
[INFO] (vectoria) This package is up to date.
[INFO] (vectorial) This package is up to date.
[INFO] (vectorio) Data has been updated for this package.
[INFO] (vectorize) This package is up to date.
[INFO] (vectorizedsampleentropy) This package is up to date.
[INFO] (vectorkit) Data has been updated for this package.
[INFO] (vectormath) This package is up to date.
[INFO] (vectorpack) This package is up to date.
[INFO] (vectorpy) This package is up to date.
[INFO] (vectors) This package is up to date.
[INFO] (vectors-of-locally-aggregate-concepts) This package is up to date.
[INFO] (vectors32) This package is up to date.
[INFO] (vectorspector) This package is up to date.
[INFO] (vectortile) This package is up to date.
[INFO] (vectortools) This package is up to date.
[INFO] (vectortween) This package is up to date.
[INFO] (vectorvondoom) This package is up to date.
[INFO] (vectotools) This package is up to date.
[INFO] (vectra-api-client) This package is up to date.
[INFO] (vectra-api-tools) This package is up to date.
[INFO] (vecvis) This package is up to date.
[INFO] (vedadb) Data has been updated for this package.
[INFO] (vedavaapi-client) This package is up to date.
[INFO] (vediccitynames) This package is up to date.
[INFO] (vedicmultiplysuhas) Data has been updated for this package.
[INFO] (vedis) This package is up to date.
[INFO] (vedit) This package is up to date.
[ERROR] (vedro) This package does not have downloadable releases.
[ERROR] (vee) This package does not have downloadable releases.
[INFO] (veeam) Data has been updated for this package.
[INFO] (veem) Data has been updated for this package.
[INFO] (veer) This package is up to date.
[INFO] (vega) Data has been updated for this package.
[INFO] (vega-datasets) Data has been updated for this package.
[ERROR] (vega-demo) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vega3) This package is up to date.
[INFO] (vegademo) Data has been updated for this package.
[INFO] (vegans) This package is up to date.
[INFO] (vegas) Data has been updated for this package.
[INFO] (vegas-lattice) This package is up to date.
[INFO] (vegascope) Data has been updated for this package.
[INFO] (vegawidget) This package is up to date.
[ERROR] (vegemite) This package does not have downloadable releases.
[INFO] (vegetablesclassify) Data has been updated for this package.
[INFO] (vegindex) This package is up to date.
[INFO] (vegu-pandax) Data has been updated for this package.
[INFO] (veh) This package is up to date.
[INFO] (vehicle-makes) Data has been updated for this package.
[INFO] (vehicular) This package is up to date.
[INFO] (veho) Data has been updated for this package.
[INFO] (veidt) This package is up to date.
[ERROR] (veil) This package does not have downloadable releases.
[INFO] (vel) This package is up to date.
[INFO] (velaris) This package is up to date.
[INFO] (velarislib) This package is up to date.
[INFO] (velbot) This package is up to date.
[INFO] (velcro) This package is up to date.
[INFO] (velenxc) This package is up to date.
[INFO] (veliberator) This package is up to date.
[INFO] (vell) This package is up to date.
[INFO] (vellichor) This package is up to date.
[INFO] (vellum) This package is up to date.
[ERROR] (velo) This package does not have downloadable releases.
[INFO] (velo-python) Data has been updated for this package.
[INFO] (velociraptor) Data has been updated for this package.
[INFO] (velocity) Data has been updated for this package.
[INFO] (velociwrapper) This package is up to date.
[INFO] (velocyto) This package is up to date.
[INFO] (velox) This package is up to date.
[INFO] (velplot) Data has been updated for this package.
[INFO] (velruse) This package is up to date.
[INFO] (velruse-naver) This package is up to date.
[INFO] (velu) This package is up to date.
[INFO] (velvet) This package is up to date.
[INFO] (veml6075) Data has been updated for this package.
[INFO] (ven) This package is up to date.
[ERROR] (venafi-openstack-heat-plugin) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (venc) This package is up to date.
[INFO] (vencode) This package is up to date.
[INFO] (vend) This package is up to date.
[INFO] (vendor) This package is up to date.
[INFO] (vendoring) Data has been updated for this package.
[INFO] (vendorize) This package is up to date.
[INFO] (vendpos) This package is up to date.
[INFO] (vendy) Data has been updated for this package.
[ERROR] (veneer) This package does not have downloadable releases.
[INFO] (vengeance) Data has been updated for this package.
[INFO] (venim) Data has been updated for this package.
[INFO] (venkkat-math) This package is up to date.
[INFO] (venkman) This package is up to date.
[INFO] (venmo) This package is up to date.
[INFO] (venmo-split) This package is up to date.
[INFO] (venmosplit) This package is up to date.
[INFO] (venn) This package is up to date.
[INFO] (venn-places) This package is up to date.
[INFO] (venom) This package is up to date.
[INFO] (venompy) This package is up to date.
[INFO] (venomseq) This package is up to date.
[INFO] (venorm) This package is up to date.
[INFO] (venstarcolortouch) Data has been updated for this package.
[INFO] (vent) Data has been updated for this package.
[INFO] (ventana) This package is up to date.
[INFO] (ventmap) Data has been updated for this package.
[INFO] (ventprogramm) Data has been updated for this package.
[INFO] (ventraip-vip-client) This package is up to date.
[INFO] (ventriloquist) This package is up to date.
[INFO] (ventsmart) Data has been updated for this package.
[INFO] (ventura) This package is up to date.
[ERROR] (venturer) This package does not have downloadable releases.
[INFO] (venturocket) This package is up to date.
[INFO] (ventx-aws-cdk-library) Data has been updated for this package.
[INFO] (venum) This package is up to date.
[INFO] (venus) This package is up to date.
[INFO] (venusian) Data has been updated for this package.
[INFO] (venusianconfiguration) This package is up to date.
[INFO] (venusifork) This package is up to date.
[INFO] (venuspy) Data has been updated for this package.
[INFO] (venv-clean) Data has been updated for this package.
[INFO] (venv-dependencies) This package is up to date.
[INFO] (venv-easy) This package is up to date.
[INFO] (venv-pack) This package is up to date.
[INFO] (venv-run) This package is up to date.
[INFO] (venv-tools) Data has been updated for this package.
[INFO] (venv-update) This package is up to date.
[INFO] (venvctrl) Data has been updated for this package.
[INFO] (venvdotapp) Data has been updated for this package.
[INFO] (venvgit2) Data has been updated for this package.
[INFO] (venvm) This package is up to date.
[INFO] (venvs) Data has been updated for this package.
[INFO] (venvstarter) This package is up to date.
[INFO] (venvtools) Data has been updated for this package.
[INFO] (venvui) This package is up to date.
[INFO] (venvupdate) This package is up to date.
[INFO] (vep-core) This package is up to date.
[ERROR] (veppy) This package does not have downloadable releases.
[INFO] (ver) This package is up to date.
[INFO] (vera) This package is up to date.
[INFO] (verace) This package is up to date.
[INFO] (veracitools) Data has been updated for this package.
[INFO] (veracitysdk) Data has been updated for this package.
[INFO] (veracode-api-signing) Data has been updated for this package.
[INFO] (veracode-python) Data has been updated for this package.
[INFO] (veracodestats) This package is up to date.
[INFO] (veracodetocsv) This package is up to date.
[INFO] (veracross-api) This package is up to date.
[INFO] (verarandom) This package is up to date.
[INFO] (verata) This package is up to date.
[ERROR] (verb-conjugate-fr) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (verb-counter) This package is up to date.
[INFO] (verbaendeliste-bundestag) This package is up to date.
[INFO] (verbalexpressions) This package is up to date.
[INFO] (verbecc) Data has been updated for this package.
[INFO] (verbose-ls) Data has been updated for this package.
[INFO] (verbose-octo-goggles) This package is up to date.
[INFO] (verboselib) This package is up to date.
[INFO] (verboselogs) This package is up to date.
[INFO] (verbosity) This package is up to date.
[ERROR] (verbula) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (verchew) Data has been updated for this package.
[INFO] (verchy) This package is up to date.
[INFO] (verconchk) This package is up to date.
[INFO] (verdb) This package is up to date.
[INFO] (verde) Data has been updated for this package.
[INFO] (verdict) This package is up to date.
[INFO] (verdigris-mu-splat) This package is up to date.
[INFO] (verdoc) Data has been updated for this package.
[INFO] (verfun) Data has been updated for this package.
[INFO] (verge) This package is up to date.
[INFO] (vergeml) This package is up to date.
[INFO] (vergil) This package is up to date.
[INFO] (vergit) This package is up to date.
[INFO] (verhaal) This package is up to date.
[INFO] (verhawk) This package is up to date.
[INFO] (vericoin) This package is up to date.
[INFO] (vericred-client) This package is up to date.
[INFO] (veridu-python) This package is up to date.
[INFO] (verif) Data has been updated for this package.
[INFO] (verifai-sdk) Data has been updated for this package.
[ERROR] (verificala) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (verification-code) This package is up to date.
[INFO] (verifier) This package is up to date.
[INFO] (verifone) Data has been updated for this package.
[INFO] (veriform) This package is up to date.
[INFO] (verify) This package is up to date.
[INFO] (verify-email) Data has been updated for this package.
[INFO] (verify-version-spec) This package is up to date.
[INFO] (verifycode) This package is up to date.
[INFO] (verifyid) This package is up to date.
[INFO] (verifykit) Data has been updated for this package.
[INFO] (verifyreq) This package is up to date.
[INFO] (verigator) This package is up to date.
[INFO] (verihash) This package is up to date.
[INFO] (verilog-vcd) This package is up to date.
[INFO] (veriloggen) Data has been updated for this package.
[INFO] (veriloglintbear) This package is up to date.
[INFO] (verilogparser) This package is up to date.
[INFO] (veriocheck) This package is up to date.
[INFO] (veripeditus) This package is up to date.
[INFO] (veriport-dashboard) This package is up to date.
[INFO] (veripress) This package is up to date.
[INFO] (veripupil) This package is up to date.
[INFO] (veripy) This package is up to date.
[INFO] (veripypi) This package is up to date.
[ERROR] (veripyx) Could not extract data from this package. See below for details.
".html" extension not supported.
[INFO] (veriservice) This package is up to date.
[INFO] (verisk) This package is up to date.
[INFO] (verispy) Data has been updated for this package.
[INFO] (veritable) This package is up to date.
[INFO] (veritas) This package is up to date.
[INFO] (veritaseum) This package is up to date.
[INFO] (veriteem) This package is up to date.
[INFO] (veriteemcomplianceledger) This package is up to date.
[INFO] (veriteos) This package is up to date.
[INFO] (veriteos-eth-abi) This package is up to date.
[INFO] (veriteos-web3) This package is up to date.
[INFO] (veritorch) Data has been updated for this package.
[INFO] (veritranspay) Data has been updated for this package.
[INFO] (verity) This package is up to date.
[INFO] (verity-sdk) Data has been updated for this package.
[INFO] (veriusapigateway) Data has been updated for this package.
[INFO] (verkhovna-rada-ukr-pl) This package is up to date.
[INFO] (verkkomaksut) This package is up to date.
[INFO] (verktyg) This package is up to date.
[INFO] (verktyg-server) This package is up to date.
[INFO] (verktyg-sqlalchemy) This package is up to date.
[INFO] (verlanize) This package is up to date.
[INFO] (verlib) This package is up to date.
[INFO] (verloopcontext) Data has been updated for this package.
[INFO] (verman) This package is up to date.
[INFO] (verme-logs) Data has been updated for this package.
[INFO] (vermin) Data has been updated for this package.
[INFO] (verminator) Data has been updated for this package.
[INFO] (vermouth) Data has been updated for this package.
[INFO] (vernam) This package is up to date.
[INFO] (vernierpygatt) This package is up to date.
[INFO] (vernum) This package is up to date.
[INFO] (vero) This package is up to date.
[INFO] (verobject) This package is up to date.
[INFO] (veros) Data has been updated for this package.
[INFO] (veros-bgc) Data has been updated for this package.
[INFO] (veroviz) Data has been updated for this package.
[INFO] (verparser) This package is up to date.
[INFO] (vers) This package is up to date.
[INFO] (versa) Data has been updated for this package.
[INFO] (versatility) This package is up to date.
[INFO] (verschemes) This package is up to date.
[INFO] (verse) This package is up to date.
[INFO] (versescraper) This package is up to date.
[INFO] (versie) This package is up to date.
[INFO] (versify) This package is up to date.
[INFO] (versign) Data has been updated for this package.
[INFO] (versile-python2) This package is up to date.
[INFO] (versile-python3) This package is up to date.
[INFO] (versio) This package is up to date.
[INFO] (version) This package is up to date.
[INFO] (version-ci-bot) This package is up to date.
[INFO] (version-control) Data has been updated for this package.
[INFO] (version-de) This package is up to date.
[INFO] (version-dem) This package is up to date.
[INFO] (version-demo) This package is up to date.
[INFO] (version-filter) This package is up to date.
[INFO] (version-hunter) This package is up to date.
[INFO] (version-information) This package is up to date.
[INFO] (version-manager) This package is up to date.
[INFO] (version-merger) This package is up to date.
[INFO] (version-parser) Data has been updated for this package.
[INFO] (version-probe) This package is up to date.
[INFO] (version-query) Data has been updated for this package.
[INFO] (version-stamp) Data has been updated for this package.
[INFO] (version-string) This package is up to date.
[INFO] (version-utils) This package is up to date.
[INFO] (version4plos) Data has been updated for this package.
[INFO] (versionah) This package is up to date.
[INFO] (versionalchemy) Data has been updated for this package.
[INFO] (versionary) This package is up to date.
[INFO] (versionbump) This package is up to date.
[INFO] (versionbumpr) Data has been updated for this package.
[INFO] (versioncheck) This package is up to date.
[INFO] (versionchecker) Data has been updated for this package.
[INFO] (versioneer) This package is up to date.
[INFO] (versioneer-518) Data has been updated for this package.
[INFO] (versioneer2) This package is up to date.
[INFO] (versioner) This package is up to date.
[INFO] (versioner-cli) This package is up to date.
[INFO] (versioneye-slack) This package is up to date.
[INFO] (versionfinder) Data has been updated for this package.
[INFO] (versionflow) Data has been updated for this package.
[INFO] (versioning) This package is up to date.
[INFO] (versioning-fs) This package is up to date.
[INFO] (versionner) Data has been updated for this package.
[INFO] (versionone-cli-sproutloud) This package is up to date.
[INFO] (versionone-to-jira-reflector) This package is up to date.
[ERROR] (versionpick) This package does not have downloadable releases.
[INFO] (versionpro) Data has been updated for this package.
[INFO] (versions) This package is up to date.
[INFO] (versions-client) This package is up to date.
[INFO] (versiontag) This package is up to date.
[INFO] (versiontools) This package is up to date.
[INFO] (versiontracker) Data has been updated for this package.
[INFO] (versionup) This package is up to date.
[INFO] (versionutil) This package is up to date.
[INFO] (verslilietuva-driver) This package is up to date.
[INFO] (verslilietuvadb) This package is up to date.
[INFO] (verst-pants-docker) This package is up to date.
[INFO] (verst-pants-s3cache) This package is up to date.
[INFO] (versuchung) This package is up to date.
[INFO] (vert) This package is up to date.
[INFO] (verta) Data has been updated for this package.
[INFO] (vertex) This package is up to date.
[INFO] (vertex2tex) Data has been updated for this package.
[INFO] (vertexarray) This package is up to date.
[INFO] (vertica-ml-python) Data has been updated for this package.
[INFO] (vertica-python) Data has been updated for this package.
[INFO] (vertica-sqlalchemy) This package is up to date.
[INFO] (vertica-sqlalchemy-0-2) This package is up to date.
[INFO] (vertica-swift-backup) This package is up to date.
[INFO] (verticaltable) This package is up to date.
[INFO] (verticapython) This package is up to date.
[INFO] (vertices) Data has been updated for this package.
[INFO] (vertigo) This package is up to date.
[INFO] (verto) This package is up to date.
[ERROR] (vertoo) This package does not have downloadable releases.
[INFO] (vertool) Data has been updated for this package.
[INFO] (vertx-eventbus) This package is up to date.
[INFO] (vertx-python-client) Data has been updated for this package.
[INFO] (veruca) This package is up to date.
[INFO] (verve-flake8-mock) This package is up to date.
[INFO] (very-basic-sql-builder) This package is up to date.
[INFO] (very-good-setuptools-git-version) This package is up to date.
[INFO] (very-plot) Data has been updated for this package.
[INFO] (very-simplexml) This package is up to date.
[INFO] (veryfay) This package is up to date.
[INFO] (verylargebits) This package is up to date.
[ERROR] (verynicelibrary) This package does not have downloadable releases.
[INFO] (veryprettytable) This package is up to date.
[INFO] (veryscrape) This package is up to date.
[INFO] (verysimplestats) This package is up to date.
[INFO] (ves) This package is up to date.
[INFO] (vesna-alhauthproxy) This package is up to date.
[INFO] (vesna-alhtools) This package is up to date.
[INFO] (vesna-spectrumsensor) This package is up to date.
[INFO] (vesnicalib) Data has been updated for this package.
[INFO] (vespa) This package is up to date.
[INFO] (vespa-slim) This package is up to date.
[INFO] (vespa-suite) Data has been updated for this package.
[INFO] (vespasian) Data has been updated for this package.
[INFO] (vespene) This package is up to date.
[INFO] (vesper) This package is up to date.
[INFO] (vessel) Data has been updated for this package.
[INFO] (vessel-scoring) This package is up to date.
[ERROR] (vesta) This package does not have downloadable releases.
[INFO] (vesta-integration-tests) This package is up to date.
[INFO] (vestarestpackage) This package is up to date.
[INFO] (vestaservice) This package is up to date.
[ERROR] (vestige) This package does not have downloadable releases.
[INFO] (vesync) This package is up to date.
[INFO] (vesync-outlet) This package is up to date.
[INFO] (veta-python-novaclient-ext) This package is up to date.
[INFO] (vetoes) This package is up to date.
[INFO] (vetrr) This package is up to date.
[INFO] (veusz) Data has been updated for this package.
[INFO] (vev) This package is up to date.
[INFO] (vevare) This package is up to date.
[INFO] (vex) This package is up to date.
[INFO] (vexaiml) This package is up to date.
[INFO] (vexatapi) This package is up to date.
[INFO] (vexbi) This package is up to date.
[INFO] (vexbot) This package is up to date.
[INFO] (vexdb) This package is up to date.
[INFO] (vexer) This package is up to date.
[INFO] (vexmessage) This package is up to date.
[INFO] (vexmpp) This package is up to date.
[INFO] (vexparser) This package is up to date.
[INFO] (vexshell) This package is up to date.
[INFO] (vexstorage) This package is up to date.
[INFO] (vext) This package is up to date.
[INFO] (vext-dbus) This package is up to date.
[INFO] (vext-gi) This package is up to date.
[INFO] (vext-libtorrent) This package is up to date.
[ERROR] (vext-opencv) This package does not have downloadable releases.
[INFO] (vext-panda3d) This package is up to date.
[INFO] (vext-pygame) This package is up to date.
[INFO] (vext-pygtk) This package is up to date.
[INFO] (vext-pyqt4) This package is up to date.
[INFO] (vext-pyqt5) This package is up to date.
[INFO] (vext-soapysdr) This package is up to date.
[INFO] (vext-vtk) This package is up to date.
[INFO] (vext-wx) This package is up to date.
[INFO] (vezda) Data has been updated for this package.
[INFO] (vf-1) This package is up to date.
[INFO] (vf-portalytics) Data has been updated for this package.
[ERROR] (vf-product-creator) Could not extract data from this package. See below for details.
Unsupported package directory structure.
[INFO] (vf3d) Data has been updated for this package.
[INFO] (vfacenet) This package is up to date.
[INFO] (vfb2ufo3) Data has been updated for this package.
[INFO] (vfclust) This package is up to date.
[INFO] (vfcode) This package is up to date.
[INFO] (vfd) This package is up to date.
[INFO] (vfdbquery) This package is up to date.
[INFO] (vfi) Data has been updated for this package.
[INFO] (vfk-msb-client-library-python) This package is up to date.
[INFO] (vfxnaming) Data has been updated for this package.
[INFO] (vfxt) Data has been updated for this package.
[INFO] (vfxtest) Data has been updated for this package.
[INFO] (vfxwindow) Data has been updated for this package.
[INFO] (vg) Data has been updated for this package.
[INFO] (vga) This package is up to date.
[INFO] (vgengine) This package is up to date.
[INFO] (vgg-gd-sso) This package is up to date.
[INFO] (vggish-keras) Data has been updated for this package.
[INFO] (vgio) Data has been updated for this package.
[INFO] (vgit) This package is up to date.
[INFO] (vgkits-vanguard) This package is up to date.
[INFO] (vgmdb) This package is up to date.
[INFO] (vgmdb-wrapper) This package is up to date.
[INFO] (vgpapi) Data has been updated for this package.
[INFO] (vgpedestrian) Data has been updated for this package.
[INFO] (vgplaycli) Data has been updated for this package.
[INFO] (vgram) This package is up to date.
[INFO] (vgrepo) This package is up to date.
[INFO] (vgrid-jupyter) Data has been updated for this package.
[INFO] (vgridpy) Data has been updated for this package.
[INFO] (vgs-cli) Data has been updated for this package.
[ERROR] (vgs-messages) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vgstash) This package is up to date.
[INFO] (vhcreator) This package is up to date.
[INFO] (vhdeps) Data has been updated for this package.
[INFO] (vhdllintbear) This package is up to date.
[INFO] (vhdmmio) Data has been updated for this package.
[INFO] (vhdre) This package is up to date.
[INFO] (vhflex) This package is up to date.
[INFO] (vhlsrs) Data has been updated for this package.
[INFO] (vhm-client) This package is up to date.
[INFO] (vhost) This package is up to date.
[INFO] (vhostino) This package is up to date.
[INFO] (vhostm) This package is up to date.
[INFO] (vhpi) This package is up to date.
[INFO] (vht) This package is up to date.
[INFO] (vhtemplate) This package is up to date.
[INFO] (vhunter) This package is up to date.
[INFO] (vhybridize) This package is up to date.
[ERROR] (vhysug) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vhysug01) This package is up to date.
[INFO] (vi) This package is up to date.
[INFO] (vi-of-first-heard) This package is up to date.
[INFO] (vi3o) Data has been updated for this package.
[INFO] (via) This package is up to date.
[INFO] (via-xterm) This package is up to date.
[INFO] (viablimp) This package is up to date.
[INFO] (viacep) This package is up to date.
[INFO] (viacoin-scrypt) This package is up to date.
[INFO] (viadict) This package is up to date.
[INFO] (viaenv) Data has been updated for this package.
[INFO] (vial) This package is up to date.
[INFO] (vialib) Data has been updated for this package.
[INFO] (viapy) This package is up to date.
[INFO] (vibe) This package is up to date.
[INFO] (vibe-analyser) This package is up to date.
[INFO] (vibe-recorder) This package is up to date.
[INFO] (viberbot) This package is up to date.
[INFO] (vibermsg) This package is up to date.
[INFO] (vibes) This package is up to date.
[ERROR] (vibgyor) This package does not have downloadable releases.
[INFO] (vibhu4agarwal) Data has been updated for this package.
[ERROR] (vibootstrap) This package does not have downloadable releases.
[INFO] (vibora) This package is up to date.
[INFO] (vibra) Data has been updated for this package.
[INFO] (vibranium) This package is up to date.
[INFO] (vibration-toolbox) Data has been updated for this package.
[INFO] (vibrations) This package is up to date.
[INFO] (vibrationtesting) This package is up to date.
[INFO] (vic) This package is up to date.
[ERROR] (vic-assistant) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vic-pyramid) This package is up to date.
[INFO] (vic-txzmq) This package is up to date.
[INFO] (vica) This package is up to date.
[INFO] (vicar2png) This package is up to date.
[INFO] (vice-outbound) This package is up to date.
[INFO] (vice-plone-outbound) This package is up to date.
[INFO] (vice-zope2-outbound) This package is up to date.
[INFO] (vicero) This package is up to date.
[INFO] (viceroy) This package is up to date.
[INFO] (vici) Data has been updated for this package.
[ERROR] (vicious) This package does not have downloadable releases.
[INFO] (vickitrix) This package is up to date.
[INFO] (vickykct) This package is up to date.
[INFO] (vicopo) This package is up to date.
[INFO] (vicorrect) Data has been updated for this package.
[INFO] (vicses-viper) This package is up to date.
[INFO] (victims-web) This package is up to date.
[INFO] (victor) This package is up to date.
[INFO] (victorpy) This package is up to date.
[INFO] (victortest) This package is up to date.
[INFO] (vid2frame) This package is up to date.
[ERROR] (vid2gif) This package does not have downloadable releases.
[INFO] (vid2mp3) This package is up to date.
[INFO] (vida) This package is up to date.
[ERROR] (vidag) This package does not have downloadable releases.
[INFO] (vidal-api-py) This package is up to date.
[INFO] (vidcutter) This package is up to date.
[INFO] (videnc) Data has been updated for this package.
[INFO] (video-behavior-tracking) This package is up to date.
[INFO] (video-brain) This package is up to date.
[INFO] (video-converter) This package is up to date.
[INFO] (video-curation) This package is up to date.
[INFO] (video-cut) This package is up to date.
[INFO] (video-facenet) Data has been updated for this package.
[INFO] (video-funnel) Data has been updated for this package.
[INFO] (video-indexer) This package is up to date.
[INFO] (video-kf) Data has been updated for this package.
[INFO] (video-konverter) Data has been updated for this package.
[INFO] (video-pipeline) This package is up to date.
[INFO] (video-streamer) This package is up to date.
[INFO] (video-to-ascii) Data has been updated for this package.
[INFO] (video-utils) Data has been updated for this package.
[INFO] (video2chars) This package is up to date.
[INFO] (video2tfrecord) This package is up to date.
[INFO] (videoanalysis) This package is up to date.
[INFO] (videoboard) This package is up to date.
[INFO] (videocloud) Data has been updated for this package.
[INFO] (videocr) Data has been updated for this package.
[INFO] (videocrawler) This package is up to date.
[INFO] (videodigest) This package is up to date.
[INFO] (videodownloader) This package is up to date.
[INFO] (videodrome) This package is up to date.
[INFO] (videofeatures) This package is up to date.
[INFO] (videoflow) Data has been updated for this package.
[INFO] (videoframes) Data has been updated for this package.
[INFO] (videogrep) This package is up to date.
[INFO] (videoguys) This package is up to date.
[INFO] (videoinfo) Data has been updated for this package.
[ERROR] (videolog) This package does not have downloadable releases.
[INFO] (videomaker) This package is up to date.
[INFO] (videomap) Data has been updated for this package.
[INFO] (videomass) Data has been updated for this package.
[INFO] (videopy) This package is up to date.
[INFO] (videoscene) This package is up to date.
[INFO] (videoscreen) This package is up to date.
[INFO] (videosequence) This package is up to date.
[INFO] (videoshot) This package is up to date.
[INFO] (videoslicer) This package is up to date.
[INFO] (videosplit) This package is up to date.
[INFO] (videotoframes) Data has been updated for this package.
[INFO] (videotools) Data has been updated for this package.
[INFO] (videotosmi) This package is up to date.
[INFO] (videotosmi-server) This package is up to date.
[INFO] (videototext) Data has been updated for this package.
[INFO] (videoutils) Data has been updated for this package.
[INFO] (vidgear) Data has been updated for this package.
[INFO] (vidhdr) This package is up to date.
[INFO] (vidhub-control) This package is up to date.
[INFO] (vidi3d) Data has been updated for this package.
[INFO] (vidjon-messenger-api) This package is up to date.
[INFO] (vidl) Data has been updated for this package.
[INFO] (vidmaster) This package is up to date.
[INFO] (vidme) This package is up to date.
[INFO] (vidme-py) This package is up to date.
[INFO] (vido) This package is up to date.
[INFO] (vidpy) This package is up to date.
[INFO] (vidscraper) This package is up to date.
[INFO] (vidsrc) Data has been updated for this package.
[INFO] (vidstab) Data has been updated for this package.
[INFO] (vidtag) Data has been updated for this package.
[INFO] (vidtext) This package is up to date.
[ERROR] (viennagrid-python) This package does not have downloadable releases.
[INFO] (viennaptm) Data has been updated for this package.
[INFO] (viessmann-data-audit) This package is up to date.
[INFO] (viesvatcheck) This package is up to date.
[INFO] (view) This package is up to date.
[INFO] (view-my-photos) This package is up to date.
[INFO] (view-spark-timeline) This package is up to date.
[INFO] (viewasuser) Data has been updated for this package.
[INFO] (viewflow) This package is up to date.
[INFO] (viewflow-extensions) This package is up to date.
[ERROR] (viewlet) This package does not have downloadable releases.
[INFO] (viewlist) This package is up to date.
[INFO] (viewlog) This package is up to date.
[INFO] (viewlogger) This package is up to date.
[INFO] (viewm) This package is up to date.
[INFO] (viewmodel) This package is up to date.
[INFO] (viewmydb) This package is up to date.
[INFO] (viewnudger) This package is up to date.
[INFO] (viewpoint) This package is up to date.
[INFO] (views) This package is up to date.
[INFO] (viewscad) This package is up to date.
[INFO] (viewstate) This package is up to date.
[INFO] (viewtable) This package is up to date.
[ERROR] (vif-plug-hw-veb) This package does not have downloadable releases.
[ERROR] (viff) This package does not have downloadable releases.
[ERROR] (viffbench) This package does not have downloadable releases.
[INFO] (vificov) This package is up to date.
[INFO] (vig-py3) This package is up to date.
[INFO] (viga) Data has been updated for this package.
[INFO] (vigenere-cipher-package2) This package is up to date.
[INFO] (vigenerecipher) This package is up to date.
[INFO] (vigil) This package is up to date.
[INFO] (vigil-cli) This package is up to date.
[INFO] (vigilance) This package is up to date.
[INFO] (vigilancemeteo) This package is up to date.
[INFO] (vigimeteo) This package is up to date.
[INFO] (vigimeteo-cli) This package is up to date.
[INFO] (vignesh-first) This package is up to date.
[INFO] (vignesh1) This package is up to date.
[INFO] (vignette) Data has been updated for this package.
[INFO] (vigo) This package is up to date.
[INFO] (vigpy3) This package is up to date.
[INFO] (vii-sdk) Data has been updated for this package.
[INFO] (viivakoodi) This package is up to date.
[ERROR] (vijaydeep) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (viki) This package is up to date.
[INFO] (viki-fabric-helpers) This package is up to date.
[ERROR] (vikram-ojha) This package does not have downloadable releases.
[INFO] (vilas) This package is up to date.
[INFO] (vildvittra) This package is up to date.
[INFO] (vilfo-api-client) Data has been updated for this package.
[INFO] (vili) This package is up to date.
[INFO] (vilib) This package is up to date.
[INFO] (villadewow) This package is up to date.
[INFO] (villagereach) This package is up to date.
[INFO] (villains) This package is up to date.
[INFO] (villainscli) This package is up to date.
[INFO] (villas-controller) This package is up to date.
[INFO] (villas-dataprocessing) Data has been updated for this package.
[INFO] (villas-node) Data has been updated for this package.
[INFO] (viltolyckor-scraper) This package is up to date.
[INFO] (vim-bridge) Data has been updated for this package.
[INFO] (vim-bridge3) This package is up to date.
[INFO] (vim-clutchify) This package is up to date.
[INFO] (vim-debug) This package is up to date.
[INFO] (vim-edit) This package is up to date.
[INFO] (vim-packadd) Data has been updated for this package.
[INFO] (vim-pck) This package is up to date.
[INFO] (vim-plugin-starter-kit) This package is up to date.
[INFO] (vim-plugins-profiler) Data has been updated for this package.
[INFO] (vim-power) This package is up to date.
[INFO] (vim-vint) This package is up to date.
[INFO] (vim2vsc) Data has been updated for this package.
[INFO] (viman) This package is up to date.
[INFO] (vimanga) This package is up to date.
[INFO] (vimap) This package is up to date.
[INFO] (vimapi) This package is up to date.
[INFO] (vimball) This package is up to date.
[ERROR] (vimbapython) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vimbuffer) Data has been updated for this package.
[INFO] (vimdatautils) Data has been updated for this package.
[INFO] (vimddr) This package is up to date.
[ERROR] (vimdown) Could not extract data from this package. See below for details.
"https://github.com/downloads/jeffbuttars/tarball/master" file could not be downloaded. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 107, in download_tar
    f.write(urlopen(url=urlesc(pkgurl), timeout=10).read())
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vimeo) This package is up to date.
[INFO] (vimeo-dl) This package is up to date.
[INFO] (vimeo-python-sdk) This package is up to date.
[INFO] (vimeo-utils) This package is up to date.
[INFO] (vimeo-wrapper) This package is up to date.
[ERROR] (vimes) This package does not have downloadable releases.
[INFO] (vimg) This package is up to date.
[INFO] (vimgolf) Data has been updated for this package.
[INFO] (vimi-ml) Data has been updated for this package.
[INFO] (vimilocfromweb) This package is up to date.
[INFO] (viminit) This package is up to date.
[ERROR] (vimiv) Could not extract data from this package. See below for details.
Execution of /usr/bin/python3.7 -m pypicontents.wrapper /tmp/pypic-extract/vimiv/vimiv-0.5.0/setup.py failed with the following messages:
Unknown reason.
Execution of /usr/bin/python2.7 -m pypicontents.wrapper /tmp/pypic-extract/vimiv/vimiv-0.5.0/setup.py failed with the following messages:
Unknown reason.Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 89, in execute_wrapper
    return json.loads(store.read()), ''
  File "/usr/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 89, in execute_wrapper
    return json.loads(store.read()), ''
  File "/usr/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

[INFO] (vimix) This package is up to date.
[INFO] (vimmock) This package is up to date.
[INFO] (vimms) Data has been updated for this package.
[INFO] (vimo-utils) Data has been updated for this package.
[INFO] (vimp) This package is up to date.
[INFO] (vimpdb) This package is up to date.
[INFO] (vimpdbhook) This package is up to date.
[INFO] (vimpy) This package is up to date.
[INFO] (vimpyre) This package is up to date.
[INFO] (vimrunner) This package is up to date.
[INFO] (vimtips) Data has been updated for this package.
[INFO] (vimtk) Data has been updated for this package.
[INFO] (vimwiki-markdown) Data has been updated for this package.
[INFO] (vimwiki-tools) This package is up to date.
[INFO] (vin-decoder) This package is up to date.
[INFO] (vin-ml) This package is up to date.
[INFO] (vin-parser) Data has been updated for this package.
[INFO] (vinayak) This package is up to date.
[INFO] (vinayak123) This package is up to date.
[INFO] (vincent) This package is up to date.
[INFO] (vincent-dxk) This package is up to date.
[INFO] (vincent-qt) This package is up to date.
[INFO] (vincenty) This package is up to date.
[INFO] (vincenty-cuda-nns) Data has been updated for this package.
[INFO] (vincepdf) Data has been updated for this package.
[INFO] (vinci) This package is up to date.
[INFO] (vincrosscheck-api-python-client) This package is up to date.
[INFO] (vinda) This package is up to date.
[INFO] (vindaloo) Data has been updated for this package.
[INFO] (vindauga) Data has been updated for this package.
[INFO] (vindi) Data has been updated for this package.
[INFO] (vindinium) This package is up to date.
[INFO] (vindta-recalk) This package is up to date.
[INFO] (vine) This package is up to date.
[INFO] (vine-python) This package is up to date.
[INFO] (vineet-ssh) This package is up to date.
[INFO] (vinepy) This package is up to date.
[INFO] (vinergy) This package is up to date.
[INFO] (vinex-ccxt) Data has been updated for this package.
[INFO] (vineyard) This package is up to date.
[INFO] (vingd) This package is up to date.
[INFO] (vinilista) This package is up to date.
[INFO] (vininfo) This package is up to date.
[INFO] (vinisto) This package is up to date.
[INFO] (vinit) This package is up to date.
[INFO] (vinli-tornado-auth) This package is up to date.
[INFO] (vinlib) This package is up to date.
[INFO] (vinnie) Data has been updated for this package.
[INFO] (vinny) This package is up to date.
[ERROR] (vino) This package does not have downloadable releases.
[INFO] (vinod-python) This package is up to date.
[INFO] (vinpack) This package is up to date.
[INFO] (vinpdf) Data has been updated for this package.
[INFO] (vins) This package is up to date.
[INFO] (vint) This package is up to date.
[INFO] (vinta-django-templated-email) This package is up to date.
[INFO] (vinta-django-waffle) This package is up to date.
[INFO] (vintage) This package is up to date.
[INFO] (vintasoftware-infomap-fork) This package is up to date.
[INFO] (vintbear) This package is up to date.
[INFO] (vinyl) This package is up to date.
[INFO] (vinyl-srw) This package is up to date.
[ERROR] (vinylang) Could not extract data from this package. See below for details.
"https://github.com/pyrated/vinyl.git/tarball/master" file could not be downloaded. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 107, in download_tar
    f.write(urlopen(url=urlesc(pkgurl), timeout=10).read())
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vinyldns-python) Data has been updated for this package.
[ERROR] (vinz) This package does not have downloadable releases.
[INFO] (viola) This package is up to date.
[INFO] (violas-client) Data has been updated for this package.
[INFO] (violent-webdriver) This package is up to date.
[INFO] (violet) This package is up to date.
[ERROR] (violets) This package does not have downloadable releases.
[INFO] (vip) This package is up to date.
[INFO] (vip-hci) Data has been updated for this package.
[INFO] (vipaccessui) This package is up to date.
[INFO] (vipc) This package is up to date.
[INFO] (vipc2) This package is up to date.
[INFO] (vipcode) This package is up to date.
[INFO] (vipe) This package is up to date.
[INFO] (viper) This package is up to date.
[INFO] (viper-framework) Data has been updated for this package.
[INFO] (viper-infra-commander) Data has been updated for this package.
[INFO] (viper-swift) This package is up to date.
[INFO] (viperdriver) Data has been updated for this package.
[INFO] (viperindeed) Data has been updated for this package.
[INFO] (viperlib) Data has been updated for this package.
[INFO] (viperpy) This package is up to date.
[INFO] (vipertools) Data has been updated for this package.
[INFO] (vipkid-common) This package is up to date.
[INFO] (vippool-storage) This package is up to date.
[INFO] (vipr-data) This package is up to date.
[INFO] (vipr-data-cli) This package is up to date.
[INFO] (vips-abi) Data has been updated for this package.
[INFO] (vips-hash) Data has been updated for this package.
[INFO] (vips-utils) Data has been updated for this package.
[INFO] (vipster) This package is up to date.
[INFO] (viptela) Data has been updated for this package.
[INFO] (viptela-python) This package is up to date.
[INFO] (vipy) Data has been updated for this package.
[INFO] (vipython) This package is up to date.
[INFO] (viral-loops-api) This package is up to date.
[INFO] (viralata) This package is up to date.
[INFO] (viraloverlay) This package is up to date.
[INFO] (viralrecall) This package is up to date.
[INFO] (viratoken) This package is up to date.
[ERROR] (vircd) This package does not have downloadable releases.
[INFO] (vire) Data has been updated for this package.
[INFO] (viren) This package is up to date.
[INFO] (vireo) This package is up to date.
[INFO] (vireosnp) Data has been updated for this package.
[INFO] (viresclient) Data has been updated for this package.
[INFO] (virga) This package is up to date.
[INFO] (virgil-crypto) Data has been updated for this package.
[INFO] (virgil-crypto-lib) Data has been updated for this package.
[INFO] (virgil-sdk) Data has been updated for this package.
[ERROR] (virgin) This package does not have downloadable releases.
[ERROR] (virgin-program) This package does not have downloadable releases.
[ERROR] (virgin1) This package does not have downloadable releases.
[INFO] (virgo) Data has been updated for this package.
[INFO] (viridian) Data has been updated for this package.
[INFO] (viridis) This package is up to date.
[INFO] (virl-utils) This package is up to date.
[INFO] (virlsdk) This package is up to date.
[INFO] (virlutils) This package is up to date.
[INFO] (viroconcom) Data has been updated for this package.
[INFO] (viron) This package is up to date.
[INFO] (virsh-start-stop) This package is up to date.
[INFO] (virshdumpparser) This package is up to date.
[INFO] (virt-back) This package is up to date.
[INFO] (virt-backup) Data has been updated for this package.
[INFO] (virt-lightning) Data has been updated for this package.
[INFO] (virt-manager-pure-cli-yen3) This package is up to date.
[INFO] (virtapi) This package is up to date.
[INFO] (virtcoilphase) Data has been updated for this package.
[ERROR] (virtdc) This package does not have downloadable releases.
[INFO] (virtenv) This package is up to date.
[INFO] (virtenviro) This package is up to date.
[INFO] (virtfs) This package is up to date.
[INFO] (virtme) This package is up to date.
[INFO] (virtparade) Data has been updated for this package.
[INFO] (virtscreen) This package is up to date.
[INFO] (virtshell-commands) This package is up to date.
[INFO] (virtstrap) This package is up to date.
[INFO] (virtstrap-bundler-and-npm) This package is up to date.
[INFO] (virtstrap-core) This package is up to date.
[INFO] (virtstrap-custom-script) This package is up to date.
[INFO] (virtstrap-local) This package is up to date.
[INFO] (virtstrap-npm) This package is up to date.
[INFO] (virtstrap-ruby-bundler) This package is up to date.
[INFO] (virtstrap-sample-plugin) This package is up to date.
[INFO] (virtual-ipm) Data has been updated for this package.
[INFO] (virtual-less) This package is up to date.
[INFO] (virtual-microbes) This package is up to date.
[INFO] (virtual-node) This package is up to date.
[INFO] (virtual-scanner) Data has been updated for this package.
[INFO] (virtual-storage-manager) This package is up to date.
[INFO] (virtual-touchpad) This package is up to date.
[INFO] (virtualbmc) Data has been updated for this package.
[INFO] (virtualbox) This package is up to date.
[INFO] (virtualchain) This package is up to date.
[ERROR] (virtualdc) This package does not have downloadable releases.
[INFO] (virtualecologist) This package is up to date.
[INFO] (virtualenv) Data has been updated for this package.
[INFO] (virtualenv-activator) This package is up to date.
[INFO] (virtualenv-api) This package is up to date.
[INFO] (virtualenv-bin) This package is up to date.
[INFO] (virtualenv-clone) This package is up to date.
[INFO] (virtualenv-commands) This package is up to date.
[INFO] (virtualenv-distribute) This package is up to date.
[INFO] (virtualenv-emacs) This package is up to date.
[INFO] (virtualenv-hax) This package is up to date.
[INFO] (virtualenv-here) This package is up to date.
[INFO] (virtualenv-mgr) This package is up to date.
[INFO] (virtualenv-multiver) This package is up to date.
[INFO] (virtualenv-relocate) This package is up to date.
[INFO] (virtualenv-rewrite) This package is up to date.
[INFO] (virtualenv-sh) This package is up to date.
[INFO] (virtualenv-switcher) This package is up to date.
[INFO] (virtualenv-tools) This package is up to date.
[INFO] (virtualenv-tools3) This package is up to date.
[INFO] (virtualenv2) This package is up to date.
[ERROR] (virtualenv3) This package does not have downloadable releases.
[INFO] (virtualenvclone) This package is up to date.
[INFO] (virtualenvconfig) Data has been updated for this package.
[INFO] (virtualenvcontext) This package is up to date.
[INFO] (virtualenvify) This package is up to date.
[INFO] (virtualenvmanager) This package is up to date.
[INFO] (virtualenvondemand) This package is up to date.
[INFO] (virtualenvrunner) This package is up to date.
[INFO] (virtualenvtools) This package is up to date.
[INFO] (virtualenvutils) This package is up to date.
[INFO] (virtualenvwrapper) This package is up to date.
[INFO] (virtualenvwrapper-alex) This package is up to date.
[INFO] (virtualenvwrapper-basic) This package is up to date.
[INFO] (virtualenvwrapper-bitbucket) This package is up to date.
[INFO] (virtualenvwrapper-codeintel) This package is up to date.
[INFO] (virtualenvwrapper-configvar) This package is up to date.
[INFO] (virtualenvwrapper-django) This package is up to date.
[INFO] (virtualenvwrapper-django-template) This package is up to date.
[INFO] (virtualenvwrapper-djangodeploy) This package is up to date.
[INFO] (virtualenvwrapper-emacs-desktop) This package is up to date.
[INFO] (virtualenvwrapper-gem) This package is up to date.
[INFO] (virtualenvwrapper-github) This package is up to date.
[INFO] (virtualenvwrapper-nekbot) This package is up to date.
[ERROR] (virtualenvwrapper-npm) This package does not have downloadable releases.
[INFO] (virtualenvwrapper-powershell) This package is up to date.
[INFO] (virtualenvwrapper-project) This package is up to date.
[INFO] (virtualenvwrapper-sublime) This package is up to date.
[INFO] (virtualenvwrapper-tmpenv) This package is up to date.
[INFO] (virtualenvwrapper-tryton) This package is up to date.
[INFO] (virtualenvwrapper-win) This package is up to date.
[INFO] (virtualenvwrapperwin) This package is up to date.
[INFO] (virtualenvy) This package is up to date.
[INFO] (virtualfish) This package is up to date.
[INFO] (virtualhermit) This package is up to date.
[INFO] (virtualhost) This package is up to date.
[INFO] (virtualizorapi) This package is up to date.
[INFO] (virtualkeyring) This package is up to date.
[INFO] (virtuallinks) This package is up to date.
[INFO] (virtualmemprotect) This package is up to date.
[INFO] (virtualmethod) This package is up to date.
[INFO] (virtualmicrobes) This package is up to date.
[INFO] (virtualmod) This package is up to date.
[INFO] (virtualpdu) This package is up to date.
[INFO] (virtualpp) Data has been updated for this package.
[ERROR] (virtualpp-abhijithneilabraham) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[ERROR] (virtualspection) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (virtualtime) This package is up to date.
[INFO] (virtualvideo) Data has been updated for this package.
[INFO] (virtue) This package is up to date.
[INFO] (virtuinbridge) This package is up to date.
[INFO] (virtuoso) This package is up to date.
[INFO] (virtworld) This package is up to date.
[ERROR] (virus) This package does not have downloadable releases.
[INFO] (virus-beta) This package is up to date.
[INFO] (virus-typer) Data has been updated for this package.
[INFO] (virustotal) This package is up to date.
[INFO] (virustotal-api) Data has been updated for this package.
[INFO] (virustotal-api-v2) This package is up to date.
[INFO] (virustotal-python) This package is up to date.
[INFO] (virustotal2) This package is up to date.
[INFO] (virustotal3) Data has been updated for this package.
[INFO] (virustotalparser) This package is up to date.
[INFO] (vis) This package is up to date.
[INFO] (vis-framework) This package is up to date.
[INFO] (vis-graph) Data has been updated for this package.
[INFO] (vis-mol2-pack) This package is up to date.
[INFO] (vis-net) This package is up to date.
[INFO] (vis-sample) This package is up to date.
[INFO] (vis2048) This package is up to date.
[INFO] (vis3d) This package is up to date.
[INFO] (visa) This package is up to date.
[INFO] (visa-api) Data has been updated for this package.
[INFO] (visaplan-js-urlsplit) This package is up to date.
[INFO] (visaplan-kitchen) This package is up to date.
[INFO] (visaplan-namespace-pkg-resource) This package is up to date.
[INFO] (visaplan-plone-adapters) This package is up to date.
[INFO] (visaplan-plone-ajaxnavigation) Data has been updated for this package.
[INFO] (visaplan-plone-animations) This package is up to date.
[INFO] (visaplan-plone-base) This package is up to date.
[INFO] (visaplan-plone-behaviors) This package is up to date.
[INFO] (visaplan-plone-breadcrumbs) This package is up to date.
[INFO] (visaplan-plone-browsers) This package is up to date.
[INFO] (visaplan-plone-groups) This package is up to date.
[INFO] (visaplan-plone-infohubs) This package is up to date.
[INFO] (visaplan-plone-interfaces) This package is up to date.
[INFO] (visaplan-plone-pdfexport) This package is up to date.
[INFO] (visaplan-plone-sqlwrapper) This package is up to date.
[INFO] (visaplan-plone-tools) Data has been updated for this package.
[INFO] (visaplan-plone-transform) This package is up to date.
[INFO] (visaplan-recipe-symlinks) This package is up to date.
[INFO] (visaplan-tools) Data has been updated for this package.
[INFO] (visar) Data has been updated for this package.
[INFO] (visaxrated) This package is up to date.
[INFO] (visbeat) This package is up to date.
[INFO] (visbrain) Data has been updated for this package.
[INFO] (visci) This package is up to date.
[INFO] (viscid) Data has been updated for this package.
[INFO] (viscm) Data has been updated for this package.
[INFO] (visdcc) This package is up to date.
[INFO] (visdebug) This package is up to date.
[INFO] (visdetect) Data has been updated for this package.
[INFO] (visdom) Data has been updated for this package.
[INFO] (visdom-helper) This package is up to date.
[INFO] (visdom-observer) This package is up to date.
[INFO] (visdom-plotly) This package is up to date.
[INFO] (visdom-pooled) This package is up to date.
[INFO] (visdom-wrapper) This package is up to date.
[INFO] (visearch) Data has been updated for this package.
[INFO] (visedit) This package is up to date.
[INFO] (visen) Data has been updated for this package.
[INFO] (visens) Data has been updated for this package.
[INFO] (vishap) This package is up to date.
[INFO] (vishnu) This package is up to date.
[INFO] (vishwakarma) Data has been updated for this package.
[ERROR] (vishwali-package) This package does not have downloadable releases.
[ERROR] (vishwamacsmodule) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vishwamtestpackage) Data has been updated for this package.
[INFO] (visibility-graph) This package is up to date.
[INFO] (visible) This package is up to date.
[INFO] (visibudget) This package is up to date.
[INFO] (visidata) This package is up to date.
[INFO] (visie) This package is up to date.
[INFO] (visigoth) Data has been updated for this package.
[INFO] (visilibity) This package is up to date.
[INFO] (visinum) This package is up to date.
[INFO] (visio2img) This package is up to date.
[INFO] (visio2pdf) This package is up to date.
[ERROR] (vision) This package does not have downloadable releases.
[ERROR] (vision-pipeline-runtime) This package does not have downloadable releases.
[INFO] (vision-tile-query) This package is up to date.
[INFO] (vision-utils) This package is up to date.
[INFO] (vision-utils-0-1-1) This package is up to date.
[INFO] (visionary) This package is up to date.
[INFO] (visionarypm) This package is up to date.
[INFO] (visioncpp) This package is up to date.
[INFO] (visionegg) This package is up to date.
[INFO] (visionkit) This package is up to date.
[INFO] (visionmadeeasy) This package is up to date.
[INFO] (visions) Data has been updated for this package.
[INFO] (visionseed) Data has been updated for this package.
[INFO] (visit) This package is up to date.
[INFO] (visitdir) This package is up to date.
[INFO] (visitoolkit-connector) This package is up to date.
[INFO] (visitoolkit-eventsystem) This package is up to date.
[INFO] (visitor) This package is up to date.
[INFO] (visitor-mark-down) This package is up to date.
[INFO] (visitors) This package is up to date.
[INFO] (visitsimv2) This package is up to date.
[INFO] (visjs2jupyter) This package is up to date.
[INFO] (viskit) This package is up to date.
[INFO] (vislib) This package is up to date.
[INFO] (vislparser) This package is up to date.
[INFO] (visma) This package is up to date.
[INFO] (vismap) Data has been updated for this package.
[INFO] (visonicalarm) This package is up to date.
[INFO] (visonicalarm2) This package is up to date.
[INFO] (visoptslider) Data has been updated for this package.
[INFO] (vispa) This package is up to date.
[INFO] (vispa-augeroffline) This package is up to date.
[INFO] (vispr) This package is up to date.
[INFO] (vispy) Data has been updated for this package.
[INFO] (visql) This package is up to date.
[INFO] (visreader) This package is up to date.
[INFO] (vistascraper) This package is up to date.
[INFO] (vistautils) Data has been updated for this package.
[INFO] (vistickedword) Data has been updated for this package.
[INFO] (vistir) Data has been updated for this package.
[INFO] (vistrails) This package is up to date.
[INFO] (vistring) This package is up to date.
[INFO] (visu) Data has been updated for this package.
[ERROR] (visual) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (visual-behavior-ophys) This package is up to date.
[INFO] (visual-genome) This package is up to date.
[INFO] (visual-hash) This package is up to date.
[INFO] (visual-logging) This package is up to date.
[INFO] (visual-midi) Data has been updated for this package.
[INFO] (visual-search-nets) This package is up to date.
[INFO] (visual-timer) This package is up to date.
[INFO] (visualcaptcha) This package is up to date.
[INFO] (visualdata) This package is up to date.
[INFO] (visualdeploy) This package is up to date.
[INFO] (visualdiff) This package is up to date.
[INFO] (visualdiscriminationtask) This package is up to date.
[INFO] (visualdl) This package is up to date.
[INFO] (visualequation) This package is up to date.
[INFO] (visualgraphdebugger-zacharyh211) This package is up to date.
[INFO] (visualise-spacy-pattern) Data has been updated for this package.
[INFO] (visualise-spacy-tree) This package is up to date.
[INFO] (visualise-spacy-tree-0-0-1) This package is up to date.
[INFO] (visualisedictionary) This package is up to date.
[INFO] (visualization) This package is up to date.
[INFO] (visualize-logs) This package is up to date.
[INFO] (visualize-ml) This package is up to date.
[INFO] (visualmath) Data has been updated for this package.
[INFO] (visualml) This package is up to date.
[INFO] (visualops) This package is up to date.
[INFO] (visualpic) This package is up to date.
[INFO] (visualportfolio) This package is up to date.
[INFO] (visualpriors) Data has been updated for this package.
[INFO] (visualproxy) This package is up to date.
[INFO] (visualqc) Data has been updated for this package.
[INFO] (visualsnoop) This package is up to date.
[INFO] (visualstudio) This package is up to date.
[INFO] (visualtitle) This package is up to date.
[ERROR] (visupedia) This package does not have downloadable releases.
[INFO] (visvalingam) This package is up to date.
[INFO] (visvalingamwyatt) This package is up to date.
[INFO] (visvis) Data has been updated for this package.
[INFO] (visvmtagger) This package is up to date.
[INFO] (vit) Data has been updated for this package.
[INFO] (vitables) Data has been updated for this package.
[INFO] (vitabs) This package is up to date.
[INFO] (vitae) This package is up to date.
[INFO] (vital-tools) This package is up to date.
[INFO] (vitalfilepy) Data has been updated for this package.
[INFO] (vitalharsharm) This package is up to date.
[INFO] (vitality) Data has been updated for this package.
[INFO] (vitalstyles) This package is up to date.
[INFO] (vitalus) This package is up to date.
[ERROR] (vitamin) This package does not have downloadable releases.
[INFO] (vitaoptimum) Data has been updated for this package.
[INFO] (vite) Data has been updated for this package.
[INFO] (vitenstools) This package is up to date.
[INFO] (viterbi-trellis) This package is up to date.
[INFO] (vitex) This package is up to date.
[INFO] (vito) Data has been updated for this package.
[INFO] (vitrage) Data has been updated for this package.
[INFO] (vitrage-dashboard) Data has been updated for this package.
[INFO] (vitrage-tempest-plugin) Data has been updated for this package.
[INFO] (vitrage-tempest-tests) This package is up to date.
[INFO] (vitriolic) Data has been updated for this package.
[INFO] (vittles) Data has been updated for this package.
[INFO] (viu) This package is up to date.
[INFO] (viv-synapse) This package is up to date.
[INFO] (viv-utils) This package is up to date.
[ERROR] (viv-vertex) This package does not have downloadable releases.
[INFO] (viva) Data has been updated for this package.
[ERROR] (vivac-utils) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vivaredis) This package is up to date.
[INFO] (vivarium) Data has been updated for this package.
[INFO] (vivarium-cluster-tools) Data has been updated for this package.
[INFO] (vivarium-inputs) Data has been updated for this package.
[INFO] (vivarium-public-health) Data has been updated for this package.
[INFO] (vivek) This package is up to date.
[ERROR] (vivekapp) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vivialconnect) Data has been updated for this package.
[INFO] (vivian) This package is up to date.
[INFO] (vivid) Data has been updated for this package.
[INFO] (vivid-db) Data has been updated for this package.
[INFO] (vivid-node) Data has been updated for this package.
[INFO] (vivid-root) Data has been updated for this package.
[INFO] (vivid-store) Data has been updated for this package.
[ERROR] (vivienne) This package does not have downloadable releases.
[INFO] (vivint-slickqa-snot) This package is up to date.
[ERROR] (vivisect) This package does not have downloadable releases.
[INFO] (vivisect-vstruct-wb) This package is up to date.
[ERROR] (vivisect2) This package does not have downloadable releases.
[INFO] (vivludo) Data has been updated for this package.
[INFO] (vivo-rdflib-sparqlstore) This package is up to date.
[INFO] (vivodict) This package is up to date.
[INFO] (vivoharvester) This package is up to date.
[INFO] (vix) This package is up to date.
[INFO] (vixen) Data has been updated for this package.
[INFO] (viytestlib) This package is up to date.
[INFO] (viz) This package is up to date.
[INFO] (vizarray) This package is up to date.
[INFO] (vizbee) This package is up to date.
[ERROR] (vizbot) This package does not have downloadable releases.
[INFO] (vizdnn) Data has been updated for this package.
[INFO] (vizdoom) This package is up to date.
[INFO] (vizer) Data has been updated for this package.
[INFO] (vizhash) This package is up to date.
[INFO] (vizic) This package is up to date.
[INFO] (vizier) This package is up to date.
[INFO] (vizier-webapi) Data has been updated for this package.
[INFO] (vizing) This package is up to date.
[INFO] (vizion-composite-key) This package is up to date.
[INFO] (vizir) This package is up to date.
[INFO] (vizmdend) This package is up to date.
[INFO] (vizme) Data has been updated for this package.
[INFO] (viznet) This package is up to date.
[INFO] (vizontele) This package is up to date.
[INFO] (vizqes) This package is up to date.
[INFO] (vizseq) Data has been updated for this package.
[INFO] (vizsequence) This package is up to date.
[INFO] (vizstack-py) Data has been updated for this package.
[INFO] (viztricks) This package is up to date.
[INFO] (vizuka) This package is up to date.
[INFO] (vizydrop-sdk) This package is up to date.
[INFO] (vj-hello) This package is up to date.
[INFO] (vjson) This package is up to date.
[INFO] (vjunit) Data has been updated for this package.
[INFO] (vjy-nester) This package is up to date.
[INFO] (vk) This package is up to date.
[INFO] (vk-advanced-api) This package is up to date.
[INFO] (vk-analyse) This package is up to date.
[INFO] (vk-api) Data has been updated for this package.
[INFO] (vk-api-boroda34) This package is up to date.
[INFO] (vk-audio) Data has been updated for this package.
[INFO] (vk-bot) This package is up to date.
[INFO] (vk-botting) Data has been updated for this package.
[INFO] (vk-client) This package is up to date.
[INFO] (vk-eleven) Data has been updated for this package.
[INFO] (vk-facenet) This package is up to date.
[INFO] (vk-handle-bot) This package is up to date.
[INFO] (vk-music) This package is up to date.
[INFO] (vk-py) Data has been updated for this package.
[INFO] (vk-requests) This package is up to date.
[INFO] (vk-scraper) Data has been updated for this package.
[INFO] (vk-songs) Data has been updated for this package.
[INFO] (vk-text-parser) This package is up to date.
[ERROR] (vk-user-analyse) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vk-wall-poster) This package is up to date.
[INFO] (vk2tg) This package is up to date.
[INFO] (vk7) This package is up to date.
[INFO] (vkapi) Data has been updated for this package.
[INFO] (vkapi-rmq-client) Data has been updated for this package.
[INFO] (vkapi8) This package is up to date.
[INFO] (vkapipy) This package is up to date.
[INFO] (vkappauth) This package is up to date.
[INFO] (vkbeautify) This package is up to date.
[ERROR] (vkbot) This package does not have downloadable releases.
[INFO] (vkbottle) Data has been updated for this package.
[INFO] (vkbottletest) Data has been updated for this package.
[INFO] (vkchatbot) Data has been updated for this package.
[INFO] (vkcoin) This package is up to date.
[INFO] (vkcoinapi) This package is up to date.
[INFO] (vkconnections) This package is up to date.
[INFO] (vkd) This package is up to date.
[INFO] (vkdelivery) This package is up to date.
[INFO] (vkeasybot) This package is up to date.
[ERROR] (vkexplorer) This package does not have downloadable releases.
[INFO] (vkhealth) This package is up to date.
[INFO] (vkl) This package is up to date.
[INFO] (vklancer) This package is up to date.
[ERROR] (vklib) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vklibrary) This package is up to date.
[INFO] (vkmsg) This package is up to date.
[INFO] (vkmusic) This package is up to date.
[INFO] (vkontakte) This package is up to date.
[INFO] (vkontakte2) This package is up to date.
[INFO] (vkpoint-api) Data has been updated for this package.
[INFO] (vkpore) This package is up to date.
[INFO] (vkpy) This package is up to date.
[INFO] (vkstreaming) This package is up to date.
[INFO] (vktop) This package is up to date.
[INFO] (vkts) Data has been updated for this package.
[INFO] (vl) This package is up to date.
[INFO] (vl53l0x) This package is up to date.
[INFO] (vl53l1x) Data has been updated for this package.
[INFO] (vl53l1x2) This package is up to date.
[INFO] (vlab-api-common) This package is up to date.
[INFO] (vlab-auth-service) This package is up to date.
[INFO] (vlab-inf-common) Data has been updated for this package.
[INFO] (vlab-ipam-api) This package is up to date.
[INFO] (vlab-mgr) This package is up to date.
[INFO] (vlabtool) This package is up to date.
[INFO] (vlac) This package is up to date.
[INFO] (vladiate) Data has been updated for this package.
[INFO] (vladislav-aksentev-brain-games) Data has been updated for this package.
[INFO] (vlc-ctrl) This package is up to date.
[INFO] (vlc-helper) This package is up to date.
[INFO] (vlc-http) Data has been updated for this package.
[INFO] (vlccast) This package is up to date.
[INFO] (vlcli) This package is up to date.
[INFO] (vlcmetadata01) This package is up to date.
[INFO] (vlcp) Data has been updated for this package.
[INFO] (vlcp-docker-plugin) This package is up to date.
[INFO] (vlcp-event-cython) This package is up to date.
[INFO] (vlcplaylister) This package is up to date.
[INFO] (vlcpssh) This package is up to date.
[INFO] (vlcradio) This package is up to date.
[INFO] (vlctvstation) This package is up to date.
[INFO] (vlcyt) Data has been updated for this package.
[INFO] (vld) This package is up to date.
[INFO] (vlde) This package is up to date.
[INFO] (vlead) This package is up to date.
[INFO] (vlermv) This package is up to date.
[INFO] (vlfeat-ctypes) This package is up to date.
[ERROR] (vlgp) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vlib) This package is up to date.
[INFO] (vlibras-translate) Data has been updated for this package.
[INFO] (vlight) This package is up to date.
[INFO] (vlinx-util) This package is up to date.
[INFO] (vlk-django-jsonfield) This package is up to date.
[INFO] (vlm) Data has been updated for this package.
[INFO] (vlog) Data has been updated for this package.
[INFO] (vltools) Data has been updated for this package.
[INFO] (vlttng) This package is up to date.
[INFO] (vm) This package is up to date.
[INFO] (vm-automation) This package is up to date.
[INFO] (vm-cpf-validator) This package is up to date.
[INFO] (vm-translator) This package is up to date.
[INFO] (vm2ami) This package is up to date.
[INFO] (vm5k) This package is up to date.
[INFO] (vmaas-report) This package is up to date.
[INFO] (vmaccess) This package is up to date.
[INFO] (vmad) This package is up to date.
[INFO] (vmagent) This package is up to date.
[INFO] (vmail-manager) Data has been updated for this package.
[INFO] (vmakedk) This package is up to date.
[INFO] (vmap) This package is up to date.
[INFO] (vmapcrawler) Data has been updated for this package.
[ERROR] (vmath) This package does not have downloadable releases.
[INFO] (vmbox) This package is up to date.
[INFO] (vmbuilder) This package is up to date.
[INFO] (vmc) This package is up to date.
[INFO] (vmcenter) Data has been updated for this package.
[INFO] (vmcloak) This package is up to date.
[ERROR] (vmcontroller) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vmcontroller-common) This package is up to date.
[INFO] (vmcontroller-guest) This package is up to date.
[INFO] (vmcontroller-host) This package is up to date.
[INFO] (vmd) This package is up to date.
[ERROR] (vmd2json) This package does not have downloadable releases.
[INFO] (vmdpy) This package is up to date.
[INFO] (vmemclient) This package is up to date.
[INFO] (vmerlin) This package is up to date.
[INFO] (vmessage) This package is up to date.
[ERROR] (vmeta) This package does not have downloadable releases.
[INFO] (vmetainfo) Data has been updated for this package.
[INFO] (vmf) This package is up to date.
[INFO] (vmfactory) This package is up to date.
[INFO] (vmflib) This package is up to date.
[INFO] (vmfusion) This package is up to date.
[INFO] (vmfusion-cli) This package is up to date.
[INFO] (vmguestlib) This package is up to date.
[INFO] (vmhome) This package is up to date.
[INFO] (vmi) Data has been updated for this package.
[INFO] (vmine) This package is up to date.
[INFO] (vminspect) This package is up to date.
[INFO] (vminute) This package is up to date.
[INFO] (vmjuggler) This package is up to date.
[INFO] (vmlib) This package is up to date.
[ERROR] (vmm-alerts) This package does not have downloadable releases.
[INFO] (vmmonprobe) This package is up to date.
[INFO] (vmnlcli) Data has been updated for this package.
[INFO] (vmo) This package is up to date.
[INFO] (vmock) This package is up to date.
[INFO] (vmonere) This package is up to date.
[INFO] (vmongodb) This package is up to date.
[INFO] (vmorpheusvoice) This package is up to date.
[INFO] (vmpc) This package is up to date.
[INFO] (vmpck-distributions) Data has been updated for this package.
[INFO] (vmpooler-client) This package is up to date.
[INFO] (vmprof) This package is up to date.
[INFO] (vmprof-flamegraph) This package is up to date.
[INFO] (vmprofit) This package is up to date.
[INFO] (vmpy) This package is up to date.
[ERROR] (vmpython) Could not extract data from this package. See below for details.
".py" extension not supported.
[INFO] (vmrun-wrapper) This package is up to date.
[INFO] (vmshepherd) Data has been updated for this package.
[INFO] (vmshepherd-aws-drivers) Data has been updated for this package.
[INFO] (vmshepherd-runtime-postgres-driver) Data has been updated for this package.
[INFO] (vmshepherd-zookeeper-driver) This package is up to date.
[INFO] (vmsutils) This package is up to date.
[ERROR] (vmtk) Could not extract data from this package. See below for details.
This package download URL does not point to a valid file: "http://pypi.python.org/pypi/vmtk"
[INFO] (vmtlib) Data has been updated for this package.
[INFO] (vmtools) This package is up to date.
[INFO] (vmtp) This package is up to date.
[INFO] (vmupdate) This package is up to date.
[INFO] (vmutils) This package is up to date.
[ERROR] (vmux) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vmw-cloudinit-metadata) This package is up to date.
[INFO] (vmw-vco) This package is up to date.
[INFO] (vmw-zsi) This package is up to date.
[INFO] (vmware-backup) This package is up to date.
[INFO] (vmware-clones) This package is up to date.
[ERROR] (vmware-dvs) This package does not have downloadable releases.
[INFO] (vmware-exporter) Data has been updated for this package.
[INFO] (vmware-extract) This package is up to date.
[INFO] (vmware-nsx) Data has been updated for this package.
[ERROR] (vmware-nsx-tempest) This package does not have downloadable releases.
[INFO] (vmware-nsx-tempest-plugin) This package is up to date.
[INFO] (vmware-nsxlib) Data has been updated for this package.
[INFO] (vmware-vspc) This package is up to date.
[INFO] (vmwc) Data has been updated for this package.
[INFO] (vmwvro) This package is up to date.
[INFO] (vmwvro2) Data has been updated for this package.
[INFO] (vmxparser) Data has been updated for this package.
[INFO] (vn) This package is up to date.
[INFO] (vn-core-nlp) This package is up to date.
[INFO] (vn-oanda) This package is up to date.
[INFO] (vn-py) This package is up to date.
[ERROR] (vnc-api) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vnc2flv) This package is up to date.
[INFO] (vncauthproxy) This package is up to date.
[INFO] (vnccollab-common) This package is up to date.
[INFO] (vnccollab-content) This package is up to date.
[INFO] (vnccollab-portal) This package is up to date.
[INFO] (vnccollab-redmine) This package is up to date.
[INFO] (vnccollab-theme) This package is up to date.
[INFO] (vnccollab-zimbra) This package is up to date.
[INFO] (vncdotool) Data has been updated for this package.
[INFO] (vncorenlp) This package is up to date.
[INFO] (vnctp) Data has been updated for this package.
[INFO] (vndb-dl) This package is up to date.
[INFO] (vndb-thigh-highs) This package is up to date.
[INFO] (vne-sim) Data has been updated for this package.
[INFO] (vng-api-common) Data has been updated for this package.
[INFO] (vng-api-common-utrecht) Data has been updated for this package.
[ERROR] (vnlp) Could not extract data from this package. See below for details.
Unsupported package directory structure.
[INFO] (vnmrjpy) This package is up to date.
[INFO] (vnpy) This package is up to date.
[INFO] (vnpy-light) This package is up to date.
[INFO] (vns-web3) Data has been updated for this package.
[INFO] (vnstaty) This package is up to date.
[INFO] (vntree) Data has been updated for this package.
[INFO] (vnu-validator) Data has been updated for this package.
[ERROR] (vnuploader) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vo) This package is up to date.
[INFO] (vo-fabutils) This package is up to date.
[ERROR] (voador) This package does not have downloadable releases.
[INFO] (vobj) This package is up to date.
[INFO] (vobject) This package is up to date.
[INFO] (voc) This package is up to date.
[INFO] (voca) This package is up to date.
[INFO] (voca-gorae) This package is up to date.
[INFO] (vocab) This package is up to date.
[INFO] (vocab-builder) This package is up to date.
[ERROR] (vocab-pos-neg) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vocabs) This package is up to date.
[INFO] (vocabulary) This package is up to date.
[INFO] (vocabulary-generator) Data has been updated for this package.
[INFO] (vocabulum) Data has been updated for this package.
[INFO] (vocalist) This package is up to date.
[INFO] (vocalsalad) This package is up to date.
[ERROR] (vocasta) This package does not have downloadable releases.
[INFO] (vocball) Data has been updated for this package.
[INFO] (vocmax) Data has been updated for this package.
[INFO] (vocompr) Data has been updated for this package.
[INFO] (voctokey) This package is up to date.
[INFO] (vocuzi) This package is up to date.
[INFO] (vod-metadata) This package is up to date.
[INFO] (vod-python-sdk) This package is up to date.
[INFO] (vodacompay) Data has been updated for this package.
[INFO] (vodafone-scraper) This package is up to date.
[INFO] (vodafonem2m) Data has been updated for this package.
[ERROR] (vodc-solr) Could not extract data from this package. See below for details.
This package download URL does not point to a valid file: "http://oodt.jpl.nasa.gov/dist/vodc"
[ERROR] (vodcsite-policy) Could not extract data from this package. See below for details.
This package download URL does not point to a valid file: "http://oodt.jpl.nasa.gov/dist/vodcsite"
[ERROR] (vodcsite-theme) Could not extract data from this package. See below for details.
This package download URL does not point to a valid file: "http://oodt.jpl.nasa.gov/dist/vodcsite"
[INFO] (vodem-vodafone-k4607-z) This package is up to date.
[INFO] (vodka) Data has been updated for this package.
[INFO] (vodka-xbahn) This package is up to date.
[INFO] (vodkas) This package is up to date.
[INFO] (vodopad) This package is up to date.
[INFO] (vodscrepe) Data has been updated for this package.
[INFO] (voevent-parse) This package is up to date.
[ERROR] (voeventdb) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (voeventdb-remote) This package is up to date.
[INFO] (voeventdb-server) This package is up to date.
[INFO] (voeventlib) This package is up to date.
[INFO] (vofs) This package is up to date.
[INFO] (vogel) This package is up to date.
[INFO] (vogeler) This package is up to date.
[INFO] (voice) This package is up to date.
[INFO] (voice-assist-matthewmccann) Data has been updated for this package.
[INFO] (voice-assistant) This package is up to date.
[INFO] (voice-assistant-mm) Data has been updated for this package.
[ERROR] (voice-assistant-rk) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (voice-commander) This package is up to date.
[INFO] (voice-engine) This package is up to date.
[INFO] (voice-gender) This package is up to date.
[ERROR] (voice-manager-ar) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (voice-robotifier) This package is up to date.
[INFO] (voiceads) This package is up to date.
[INFO] (voiceatis) This package is up to date.
[ERROR] (voicebase) This package does not have downloadable releases.
[INFO] (voicebot) This package is up to date.
[INFO] (voicebox) This package is up to date.
[INFO] (voicecoding) This package is up to date.
[INFO] (voicecom) This package is up to date.
[INFO] (voicecom-sample) This package is up to date.
[INFO] (voiceinsights) This package is up to date.
[INFO] (voiceit) This package is up to date.
[INFO] (voiceit2) Data has been updated for this package.
[INFO] (voicelabs) This package is up to date.
[INFO] (voicelabs-assistant) This package is up to date.
[INFO] (voicen) Data has been updated for this package.
[INFO] (voicenotes2org) Data has been updated for this package.
[ERROR] (voicent) This package does not have downloadable releases.
[INFO] (voicent-python) This package is up to date.
[INFO] (voiceplay) This package is up to date.
[INFO] (voicepy) Data has been updated for this package.
[INFO] (voicerss-tts) This package is up to date.
[INFO] (voicetools) This package is up to date.
[INFO] (voicetotext) This package is up to date.
[INFO] (voicing) This package is up to date.
[INFO] (void) Data has been updated for this package.
[INFO] (void-object) This package is up to date.
[INFO] (void-reconstructor) This package is up to date.
[INFO] (voidpop) Data has been updated for this package.
[INFO] (voidpp-tools) This package is up to date.
[INFO] (voidpp-web-tools) This package is up to date.
[INFO] (voidwalker) This package is up to date.
[INFO] (voigtfit) Data has been updated for this package.
[INFO] (voikko) This package is up to date.
[INFO] (voila) Data has been updated for this package.
[INFO] (voila-bootstrapvue) This package is up to date.
[INFO] (voila-centered) Data has been updated for this package.
[INFO] (voila-debug) Data has been updated for this package.
[INFO] (voila-gridstack) Data has been updated for this package.
[INFO] (voila-material) Data has been updated for this package.
[INFO] (voila-plain) This package is up to date.
[INFO] (voila-reveal) Data has been updated for this package.
[INFO] (voila-vuetify) Data has been updated for this package.
[INFO] (voipms) This package is up to date.
[INFO] (voipms-python) Data has been updated for this package.
[INFO] (voitto) This package is up to date.
[INFO] (vokativ) This package is up to date.
[INFO] (vokram) This package is up to date.
[INFO] (vol) This package is up to date.
[INFO] (volapi) Data has been updated for this package.
[INFO] (volatildap) This package is up to date.
[INFO] (volatile) This package is up to date.
[INFO] (volatile-dictionary) This package is up to date.
[ERROR] (volatility) This package does not have downloadable releases.
[INFO] (volatility-profile-builder) This package is up to date.
[INFO] (volatility261) Data has been updated for this package.
[INFO] (volatility3) Data has been updated for this package.
[INFO] (volavola) Data has been updated for this package.
[INFO] (volbf) Data has been updated for this package.
[ERROR] (volcano) This package does not have downloadable releases.
[INFO] (volcano-arch) This package is up to date.
[INFO] (volcano-core) This package is up to date.
[INFO] (volcano-general) This package is up to date.
[INFO] (volcano-iec104srv) This package is up to date.
[INFO] (volcano-mbsrv) Data has been updated for this package.
[INFO] (volcano-poller) Data has been updated for this package.
[INFO] (volcano-test) This package is up to date.
[INFO] (volcano-twistedclient) This package is up to date.
[INFO] (volcano-web) This package is up to date.
[INFO] (volcasample) This package is up to date.
[INFO] (volcorner) This package is up to date.
[INFO] (voldemort) This package is up to date.
[INFO] (volgactf-final) Data has been updated for this package.
[INFO] (volgactf-final-checker-result) Data has been updated for this package.
[INFO] (volientduanlistprint) This package is up to date.
[INFO] (volition-server-management) This package is up to date.
[INFO] (volkanic) This package is up to date.
[INFO] (volkswagen) Data has been updated for this package.
[INFO] (volkswagencarnet) Data has been updated for this package.
[INFO] (volkszaehler) This package is up to date.
[INFO] (volley) Data has been updated for this package.
[INFO] (volleyup) This package is up to date.
[INFO] (vollib) This package is up to date.
[ERROR] (vollmond) This package does not have downloadable releases.
[INFO] (volly) Data has been updated for this package.
[INFO] (volmdlr) Data has been updated for this package.
[INFO] (volodymyr) This package is up to date.
[INFO] (volosapi) Data has been updated for this package.
[INFO] (volpy) Data has been updated for this package.
[INFO] (volt) This package is up to date.
[INFO] (volta) This package is up to date.
[INFO] (voltagepy) Data has been updated for this package.
[INFO] (voltcycle) This package is up to date.
[INFO] (voltha-protos) Data has been updated for this package.
[INFO] (voltools) Data has been updated for this package.
[INFO] (voltotg) Data has been updated for this package.
[INFO] (voltron) This package is up to date.
[INFO] (voltron-web) This package is up to date.
[INFO] (volume-manifest-tool) Data has been updated for this package.
[INFO] (volume2mesh) Data has been updated for this package.
[INFO] (volumetric) Data has been updated for this package.
[INFO] (volumeutil) This package is up to date.
[INFO] (volumio-buddy) This package is up to date.
[INFO] (volunteer-scheduler) This package is up to date.
[INFO] (voluptuary) This package is up to date.
[INFO] (voluptuous) Data has been updated for this package.
[INFO] (voluptuous-apispec) This package is up to date.
[INFO] (voluptuous-serialize) Data has been updated for this package.
[INFO] (voluum) This package is up to date.
[INFO] (volux) Data has been updated for this package.
[INFO] (voluxaudio) Data has been updated for this package.
[INFO] (voluxgui) Data has been updated for this package.
[INFO] (voluxlight) Data has been updated for this package.
[INFO] (volvooncall) This package is up to date.
[INFO] (vom) This package is up to date.
[INFO] (vomailer) This package is up to date.
[INFO] (voms-auth-system-openstack) This package is up to date.
[INFO] (von-agent) This package is up to date.
[INFO] (von-anchor) Data has been updated for this package.
[ERROR] (von-mises-fisher) This package does not have downloadable releases.
[ERROR] (vonapack) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vonda) This package is up to date.
[INFO] (vondrak) This package is up to date.
[ERROR] (vonk) This package does not have downloadable releases.
[INFO] (vonmises) This package is up to date.
[INFO] (vonny) This package is up to date.
[INFO] (vonx) This package is up to date.
[INFO] (voobly) Data has been updated for this package.
[INFO] (voodoo) This package is up to date.
[INFO] (voodoo-api) This package is up to date.
[INFO] (voodoo-cli) This package is up to date.
[INFO] (voodooconfig) This package is up to date.
[INFO] (vooid) This package is up to date.
[INFO] (voom) This package is up to date.
[ERROR] (vop) This package does not have downloadable releases.
[INFO] (vor) This package is up to date.
[ERROR] (vor-cycling) This package does not have downloadable releases.
[INFO] (vor12) This package is up to date.
[INFO] (vorbin) This package is up to date.
[ERROR] (vordiff) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vorlauf) This package is up to date.
[INFO] (voronoi-wall) Data has been updated for this package.
[INFO] (voropy) This package is up to date.
[INFO] (vorpal) This package is up to date.
[INFO] (vorpy) This package is up to date.
[INFO] (vorta) Data has been updated for this package.
[INFO] (vortaro) This package is up to date.
[ERROR] (vortaroj) This package does not have downloadable releases.
[INFO] (vortex) This package is up to date.
[INFO] (vortexai) This package is up to date.
[INFO] (vortexasdk) Data has been updated for this package.
[INFO] (vortexpy) Data has been updated for this package.
[INFO] (vos) Data has been updated for this package.
[INFO] (vose-alias-method) Data has been updated for this package.
[INFO] (vosk) Data has been updated for this package.
[INFO] (vostok) Data has been updated for this package.
[INFO] (vote) This package is up to date.
[INFO] (vote4film) Data has been updated for this package.
[INFO] (votebase) This package is up to date.
[INFO] (voted-ranking) This package is up to date.
[INFO] (votedperceptron) This package is up to date.
[INFO] (votes) This package is up to date.
[INFO] (voteview-dev) Data has been updated for this package.
[INFO] (voting) Data has been updated for this package.
[INFO] (voucherify) Data has been updated for this package.
[INFO] (voussoirkit) Data has been updated for this package.
[INFO] (vowcon-removal-helper) Data has been updated for this package.
[INFO] (vowel-versificon) Data has been updated for this package.
[INFO] (vowpal-porpoise) This package is up to date.
[INFO] (vowpalwabbit) Data has been updated for this package.
[ERROR] (vowpalwabbit-ds) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vowpalwabbitds) This package is up to date.
[INFO] (voxceleb-luigi) This package is up to date.
[ERROR] (voxel) This package does not have downloadable releases.
[INFO] (voxelfuse) Data has been updated for this package.
[INFO] (voxelfuse-primitives) Data has been updated for this package.
[INFO] (voxgenerator) This package is up to date.
[INFO] (voximplant-apiclient) Data has been updated for this package.
[INFO] (voximplant-client) This package is up to date.
[INFO] (voximplant-loader) This package is up to date.
[INFO] (voxjar) This package is up to date.
[INFO] (voxplex) This package is up to date.
[INFO] (voxpopuli) Data has been updated for this package.
[INFO] (voxsql) Data has been updated for this package.
[INFO] (voyage) This package is up to date.
[INFO] (voyager-icon) This package is up to date.
[INFO] (voyager-python) This package is up to date.
[INFO] (voyandz) This package is up to date.
[INFO] (voyant-indicators) Data has been updated for this package.
[INFO] (voyeur) This package is up to date.
[INFO] (voysis-python) Data has been updated for this package.
[ERROR] (vozreader) This package does not have downloadable releases.
[INFO] (vp) This package is up to date.
[INFO] (vp-nester) This package is up to date.
[INFO] (vp9dash) This package is up to date.
[INFO] (vparse) This package is up to date.
[INFO] (vpc-free) This package is up to date.
[INFO] (vpc-py) This package is up to date.
[INFO] (vpc-sh) This package is up to date.
[INFO] (vpcrouter) This package is up to date.
[INFO] (vpgtools) This package is up to date.
[INFO] (vpicwrapper) This package is up to date.
[INFO] (vping) This package is up to date.
[INFO] (vpip) Data has been updated for this package.
[INFO] (vpk) Data has been updated for this package.
[INFO] (vpl) This package is up to date.
[INFO] (vplot) Data has been updated for this package.
[INFO] (vpltest) This package is up to date.
[INFO] (vpm) This package is up to date.
[INFO] (vpn-porthole) This package is up to date.
[INFO] (vpn-slice) Data has been updated for this package.
[INFO] (vpnauth) This package is up to date.
[INFO] (vpnc) This package is up to date.
[INFO] (vpnchooser) This package is up to date.
[INFO] (vpnester) This package is up to date.
[INFO] (vpnkit) This package is up to date.
[INFO] (vpnocchio) This package is up to date.
[INFO] (vpnotebook) This package is up to date.
[INFO] (vpoller) This package is up to date.
[INFO] (vpolo) Data has been updated for this package.
[INFO] (vpopulus-api) This package is up to date.
[INFO] (vposthorde) This package is up to date.
[INFO] (vpp) Data has been updated for this package.
[INFO] (vpp-config) This package is up to date.
[INFO] (vpp-papi) This package is up to date.
[INFO] (vpplib) Data has been updated for this package.
[INFO] (vpr-netfile-parser) This package is up to date.
[INFO] (vprint) This package is up to date.
[INFO] (vprof) This package is up to date.
[INFO] (vps) This package is up to date.
[INFO] (vps-deploy) Data has been updated for this package.
[INFO] (vps-online-keeper) This package is up to date.
[INFO] (vpsearch) Data has been updated for this package.
[ERROR] (vpsql) This package does not have downloadable releases.
[INFO] (vptree) This package is up to date.
[INFO] (vpxhw-db-data-uploader) This package is up to date.
[INFO] (vpxhw-db-job-locator) This package is up to date.
[INFO] (vpxhw-ml) This package is up to date.
[INFO] (vpy) This package is up to date.
[INFO] (vpyk) This package is up to date.
[INFO] (vpython) Data has been updated for this package.
[ERROR] (vq) This package does not have downloadable releases.
[INFO] (vqsr-cnn) This package is up to date.
[INFO] (vqubo) This package is up to date.
[INFO] (vquery) This package is up to date.
[INFO] (vr) This package is up to date.
[INFO] (vr-agent) This package is up to date.
[INFO] (vr-builder) This package is up to date.
[INFO] (vr-cli) This package is up to date.
[INFO] (vr-common) This package is up to date.
[INFO] (vr-events) This package is up to date.
[INFO] (vr-imager) This package is up to date.
[INFO] (vr-launch) This package is up to date.
[INFO] (vr-runners) This package is up to date.
[INFO] (vr-server) This package is up to date.
[INFO] (vr900-connector) Data has been updated for this package.
[INFO] (vra-sdk) This package is up to date.
[INFO] (vrb) This package is up to date.
[INFO] (vrc-t70) This package is up to date.
[INFO] (vrcgal-py) This package is up to date.
[ERROR] (vrealize-6-ansible) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vrealpython-reader) This package is up to date.
[INFO] (vrecord) Data has been updated for this package.
[INFO] (vredes) This package is up to date.
[INFO] (vredis) This package is up to date.
[INFO] (vredis-py) This package is up to date.
[INFO] (vremoteapi) This package is up to date.
[INFO] (vrep-api-python) This package is up to date.
[INFO] (vrepper) This package is up to date.
[INFO] (vrequest) Data has been updated for this package.
[INFO] (vresutils) Data has been updated for this package.
[INFO] (vrijvan) This package is up to date.
[INFO] (vrmapi) This package is up to date.
[INFO] (vro-package-diff) Data has been updated for this package.
[INFO] (vroom) This package is up to date.
[INFO] (vrpc) Data has been updated for this package.
[INFO] (vrpwrp) This package is up to date.
[INFO] (vrsubtitlemaker) This package is up to date.
[INFO] (vrt-generator) Data has been updated for this package.
[INFO] (vrt-lss-field-service) Data has been updated for this package.
[INFO] (vrt-lss-lastmile) Data has been updated for this package.
[INFO] (vrt-lss-lastmile-delivery) Data has been updated for this package.
[INFO] (vrt-lss-lastmile-multidepot) Data has been updated for this package.
[INFO] (vrt-lss-merchandiser) Data has been updated for this package.
[INFO] (vrt-lss-routing) Data has been updated for this package.
[INFO] (vrt-spacy) Data has been updated for this package.
[INFO] (vrticl) Data has been updated for this package.
[INFO] (vrtneopixel) This package is up to date.
[INFO] (vrun) This package is up to date.
[INFO] (vrypy) This package is up to date.
[INFO] (vs-alias) This package is up to date.
[INFO] (vs-bootstrap-plonetheme) This package is up to date.
[INFO] (vs-bootstrap-tinymce) This package is up to date.
[INFO] (vs-contentnavigation) This package is up to date.
[INFO] (vs-contentpopup) This package is up to date.
[INFO] (vs-dashboardmanager) This package is up to date.
[INFO] (vs-emr) This package is up to date.
[INFO] (vs-event) This package is up to date.
[INFO] (vs-flexigridsearch) This package is up to date.
[INFO] (vs-genericsetup-ldap) This package is up to date.
[INFO] (vs-ip) This package is up to date.
[INFO] (vs-jquerybookmarks) This package is up to date.
[INFO] (vs-org) This package is up to date.
[INFO] (vs-tdi) This package is up to date.
[INFO] (vs-zopeskel-diazo) This package is up to date.
[INFO] (vsadmin) This package is up to date.
[ERROR] (vsajip-testdist) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vsat) This package is up to date.
[INFO] (vsbio) Data has been updated for this package.
[INFO] (vsc-base) Data has been updated for this package.
[INFO] (vsc-install) Data has been updated for this package.
[INFO] (vsc-mympirun) Data has been updated for this package.
[INFO] (vsc-utils) This package is up to date.
[INFO] (vsc-zk) This package is up to date.
[INFO] (vscext) This package is up to date.
[INFO] (vsco-scrape) This package is up to date.
[INFO] (vsco-scraper) This package is up to date.
[INFO] (vscod) Data has been updated for this package.
[INFO] (vscode-debugger-magic) This package is up to date.
[INFO] (vscode-dl) Data has been updated for this package.
[INFO] (vscode-ipynb-py-convert) This package is up to date.
[INFO] (vscodex) This package is up to date.
[INFO] (vscrapy) This package is up to date.
[INFO] (vscy-nester) This package is up to date.
[INFO] (vsdcli) This package is up to date.
[ERROR] (vsdk) This package does not have downloadable releases.
[INFO] (vse) This package is up to date.
[INFO] (vsearch) This package is up to date.
[INFO] (vserialport) This package is up to date.
[INFO] (vserver) This package is up to date.
[INFO] (vsg) Data has been updated for this package.
[INFO] (vsgan) Data has been updated for this package.
[INFO] (vsgen) This package is up to date.
[INFO] (vsgen-ptvs) Data has been updated for this package.
[INFO] (vsgui) This package is up to date.
[INFO] (vsh) Data has been updated for this package.
[INFO] (vshock) This package is up to date.
[INFO] (vsi) This package is up to date.
[INFO] (vsieve) This package is up to date.
[INFO] (vsionailibs) This package is up to date.
[INFO] (vsl) This package is up to date.
[INFO] (vslack) This package is up to date.
[ERROR] (vslmetrics) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vsm) This package is up to date.
[INFO] (vsm-dashboard) This package is up to date.
[INFO] (vsmlib) This package is up to date.
[INFO] (vsmrfs) This package is up to date.
[INFO] (vsmstreamer) This package is up to date.
[INFO] (vsn800-ardexa) This package is up to date.
[INFO] (vsnpdev) This package is up to date.
[ERROR] (vsoc) This package does not have downloadable releases.
[ERROR] (vspecs) Could not get a response from API for this package. See below for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/pypicontents/api/pypi.py", line 264, in get_pkgdata
    timeout=10).read()
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

[INFO] (vspectro) Data has been updated for this package.
[INFO] (vsphere-guest-run) This package is up to date.
[INFO] (vspheretools) This package is up to date.
[INFO] (vspider) This package is up to date.
[INFO] (vspk) Data has been updated for this package.
[INFO] (vspkgenerator) This package is up to date.
[INFO] (vsptd) This package is up to date.
[INFO] (vspyx) Data has been updated for this package.
[INFO] (vsqs) This package is up to date.
[INFO] (vsrc) This package is up to date.
[INFO] (vss-cli) Data has been updated for this package.
[INFO] (vss-python-api) This package is up to date.
[INFO] (vsscli) This package is up to date.
[INFO] (vstask) This package is up to date.
[INFO] (vstat) This package is up to date.
[INFO] (vstcompile) This package is up to date.
[INFO] (vsteamconverter) This package is up to date.
[INFO] (vstruct) This package is up to date.
[INFO] (vstruct2) This package is up to date.
[INFO] (vsts) This package is up to date.
[INFO] (vsts-cd-manager) This package is up to date.
[INFO] (vsts-cli) This package is up to date.
[INFO] (vsts-cli-admin) This package is up to date.
[INFO] (vsts-cli-admin-common) This package is up to date.
[INFO] (vsts-cli-build) This package is up to date.
[INFO] (vsts-cli-build-common) This package is up to date.
[INFO] (vsts-cli-code) This package is up to date.
[INFO] (vsts-cli-code-common) This package is up to date.
[INFO] (vsts-cli-common) This package is up to date.
[INFO] (vsts-cli-package) This package is up to date.
[INFO] (vsts-cli-package-common) This package is up to date.
[INFO] (vsts-cli-release) This package is up to date.
[INFO] (vsts-cli-release-common) This package is up to date.
[INFO] (vsts-cli-team) This package is up to date.
[INFO] (vsts-cli-team-common) This package is up to date.
[INFO] (vsts-cli-work) This package is up to date.
[INFO] (vsts-cli-work-common) This package is up to date.
[INFO] (vsts-client) This package is up to date.
[INFO] (vsts-pypi-task-demo) This package is up to date.
[INFO] (vstscdmanager) This package is up to date.
[INFO] (vststaskcmd) This package is up to date.
[INFO] (vstutils) Data has been updated for this package.
[INFO] (vsummarize) This package is up to date.
[INFO] (vsure) Data has been updated for this package.
[INFO] (vsut) This package is up to date.
[INFO] (vsutil) This package is up to date.
[INFO] (vsutillib) This package is up to date.
[INFO] (vsutillib-files) This package is up to date.
[INFO] (vsutillib-log) This package is up to date.
[INFO] (vsutillib-macos) This package is up to date.
[INFO] (vsutillib-media) This package is up to date.
[INFO] (vsutillib-mkv) This package is up to date.
[INFO] (vsutillib-network) This package is up to date.
[INFO] (vsutillib-process) This package is up to date.
[INFO] (vsutillib-pyqt) This package is up to date.
[INFO] (vsutillib-scripts) This package is up to date.
[INFO] (vsutillib-vsxml) This package is up to date.
[INFO] (vswhere) This package is up to date.
[INFO] (vswmc-cli) This package is up to date.
[INFO] (vsync) This package is up to date.
[INFO] (vsyspy) Data has been updated for this package.
[INFO] (vt) This package is up to date.
[INFO] (vt-api) This package is up to date.
[INFO] (vt-graph-api) Data has been updated for this package.
[INFO] (vt-py) Data has been updated for this package.
[INFO] (vt100) This package is up to date.
[INFO] (vt102) This package is up to date.
[INFO] (vt2geojson) This package is up to date.
[INFO] (vtapi) This package is up to date.
[INFO] (vtapi3) Data has been updated for this package.
[INFO] (vtc-scrypt) This package is up to date.
[INFO] (vtc-scrypt-new) This package is up to date.
[INFO] (vtclear) This package is up to date.
[INFO] (vtem) This package is up to date.
[INFO] (vtemplate) This package is up to date.
[INFO] (vterminal) Data has been updated for this package.
[INFO] (vtex) Data has been updated for this package.
[INFO] (vtex-client) This package is up to date.
[INFO] (vtext) Data has been updated for this package.
[INFO] (vtfunc) Data has been updated for this package.
[INFO] (vthread) This package is up to date.
[INFO] (vtiger-cloudsdk-restapi) This package is up to date.
[INFO] (vtigerpy) This package is up to date.
[INFO] (vtixypayment-assist) This package is up to date.
[INFO] (vtixyproxy) This package is up to date.
[INFO] (vtixytemplate) This package is up to date.
[INFO] (vtjp) This package is up to date.
[INFO] (vtk) This package is up to date.
[INFO] (vtk-vacreader) This package is up to date.
[INFO] (vtk-visualizer) This package is up to date.
[INFO] (vtke) This package is up to date.
[INFO] (vtki) This package is up to date.
[INFO] (vtkplotlib) Data has been updated for this package.
[INFO] (vtkplotter) Data has been updated for this package.
[INFO] (vtkviewer) Data has been updated for this package.
[INFO] (vtm) This package is up to date.
[INFO] (vto) Data has been updated for this package.
[INFO] (vtool) Data has been updated for this package.
[INFO] (vtool-ibeis) Data has been updated for this package.
[INFO] (vtools) This package is up to date.
[INFO] (vtorch) Data has been updated for this package.
[INFO] (vtp-elastic-apm) Data has been updated for this package.
[INFO] (vtpl-api) Data has been updated for this package.
[INFO] (vtpl-api-wrapper) Data has been updated for this package.
[INFO] (vtr-storages) This package is up to date.
[INFO] (vtr-utils) This package is up to date.
[INFO] (vtreat) Data has been updated for this package.
[INFO] (vtrie) This package is up to date.
[INFO] (vtrplot) This package is up to date.
[INFO] (vtrtool) This package is up to date.
[INFO] (vtscan) This package is up to date.
[INFO] (vtscanner) Data has been updated for this package.
[INFO] (vtscreatesend) This package is up to date.
[INFO] (vtsmdl) This package is up to date.
[INFO] (vtt) This package is up to date.
[INFO] (vtt-to-srt) This package is up to date.
[INFO] (vtt-to-srt3) Data has been updated for this package.
[INFO] (vttformatter) Data has been updated for this package.
[INFO] (vturesults) This package is up to date.
[INFO] (vturra) This package is up to date.
[INFO] (vtwirp) Data has been updated for this package.
[INFO] (vtwt) This package is up to date.
[INFO] (vtzero) This package is up to date.
[INFO] (vu) This package is up to date.
[INFO] (vucli) This package is up to date.
[INFO] (vudo-cmf) This package is up to date.
[INFO] (vudo-skinsetup) This package is up to date.
[INFO] (vue) This package is up to date.
[INFO] (vue-lexer) Data has been updated for this package.
[INFO] (vue4j) This package is up to date.
[INFO] (vuejspython) Data has been updated for this package.
[INFO] (vuepy) This package is up to date.
[INFO] (vuespa) Data has been updated for this package.
[INFO] (vulcan) This package is up to date.
[INFO] (vulcan-api) Data has been updated for this package.
[INFO] (vulcan-aws) Data has been updated for this package.
[INFO] (vulcan-builder) This package is up to date.
[INFO] (vulcanai) Data has been updated for this package.
[INFO] (vulcanize) This package is up to date.
[INFO] (vulcanml) This package is up to date.
[INFO] (vulcano) Data has been updated for this package.
[INFO] (vulk) This package is up to date.
[INFO] (vulkan) This package is up to date.
[INFO] (vulkbare) This package is up to date.
[INFO] (vulkn) Data has been updated for this package.
[INFO] (vulnaudit) This package is up to date.
[INFO] (vulndb) Data has been updated for this package.
[ERROR] (vulnerable) This package does not have downloadable releases.
[INFO] (vulnerable-image-check) This package is up to date.
[INFO] (vulners) Data has been updated for this package.
[INFO] (vulnix) Data has been updated for this package.
[INFO] (vulnmine) This package is up to date.
[INFO] (vulnx) This package is up to date.
[INFO] (vultr) This package is up to date.
[INFO] (vultr-speed) This package is up to date.
[INFO] (vulture) Data has been updated for this package.
[INFO] (vulturebear) This package is up to date.
[INFO] (vumasms) This package is up to date.
[INFO] (vumi) This package is up to date.
[INFO] (vumi-go) This package is up to date.
[INFO] (vumi-http-api) This package is up to date.
[INFO] (vumi-http-proxy) This package is up to date.
[INFO] (vumi-http-retry-api) This package is up to date.
[INFO] (vumi-message-store) This package is up to date.
[INFO] (vumi-msisdn-normalize-middleware) This package is up to date.
[INFO] (vumi-seidu626) Data has been updated for this package.
[INFO] (vumi-unidecode-middleware) This package is up to date.
[INFO] (vumi-wikipedia) This package is up to date.
[INFO] (vunit) This package is up to date.
[INFO] (vunit-hdl) Data has been updated for this package.
[INFO] (vunits) Data has been updated for this package.
[INFO] (vup) Data has been updated for this package.
[INFO] (vupy) This package is up to date.
[INFO] (vuray) This package is up to date.
[INFO] (vutr) This package is up to date.
[INFO] (vutwifi) This package is up to date.
[ERROR] (vuuvv) This package does not have downloadable releases.
[INFO] (vv) This package is up to date.
[INFO] (vvar) Data has been updated for this package.
[INFO] (vvcontrollers-yarmenti) Data has been updated for this package.
[INFO] (vviewer) This package is up to date.
[INFO] (vvrest) This package is up to date.
[INFO] (vvs) This package is up to date.
[INFO] (vvspy) Data has been updated for this package.
[INFO] (vvv) This package is up to date.
[INFO] (vvveb) This package is up to date.
[INFO] (vvvv) This package is up to date.
[INFO] (vvxme) Data has been updated for this package.
[INFO] (vw-gstore-adapter) This package is up to date.
[INFO] (vweb) This package is up to date.
[INFO] (vwo-python-sdk) Data has been updated for this package.
[INFO] (vwoptimize) This package is up to date.
[INFO] (vworker) This package is up to date.
[INFO] (vwr) This package is up to date.
[INFO] (vws-auth-tools) Data has been updated for this package.
[INFO] (vws-cli) Data has been updated for this package.
[INFO] (vws-python) Data has been updated for this package.
[INFO] (vws-python-mock) Data has been updated for this package.
[INFO] (vws-test-fixtures) Data has been updated for this package.
[INFO] (vx) This package is up to date.
[ERROR] (vx2008) This package does not have downloadable releases.
[INFO] (vxaat) This package is up to date.
[INFO] (vxadmin) This package is up to date.
[INFO] (vxapprouter) This package is up to date.
[INFO] (vxblastsms) This package is up to date.
[INFO] (vxcube-api) Data has been updated for this package.
[INFO] (vxdata) This package is up to date.
[INFO] (vxfreeswitch) This package is up to date.
[INFO] (vxghpages) This package is up to date.
[INFO] (vxmessenger) This package is up to date.
[INFO] (vxpolls) This package is up to date.
[ERROR] (vxqd) This package does not have downloadable releases.
[INFO] (vxrpc) This package is up to date.
[INFO] (vxsandbox) This package is up to date.
[INFO] (vxscreenshots) This package is up to date.
[INFO] (vxstreamlib) This package is up to date.
[INFO] (vxtrader) This package is up to date.
[INFO] (vxtwinio) This package is up to date.
[INFO] (vxtwitter) This package is up to date.
[INFO] (vxutils) This package is up to date.
[INFO] (vxvas2nets) This package is up to date.
[INFO] (vxwassup) This package is up to date.
[INFO] (vxyowsup) This package is up to date.
[INFO] (vy) Data has been updated for this package.
[INFO] (vyattaconfparser) Data has been updated for this package.
[INFO] (vycro) This package is up to date.
[INFO] (vydia) This package is up to date.
[INFO] (vyirc) Data has been updated for this package.
[INFO] (vyked) This package is up to date.
[INFO] (vyked-orm) This package is up to date.
[INFO] (vymgmt) This package is up to date.
[INFO] (vyper) Data has been updated for this package.
[INFO] (vyper-bot) This package is up to date.
[INFO] (vyper-config) Data has been updated for this package.
[INFO] (vyper-debug) This package is up to date.
[ERROR] (vyperlogixpyaxlib) Could not extract data from this package. See below for details.
This package download URL does not point to a valid file: "http://code.google.com/p/sfpyax/"
[ERROR] (vyrsion) This package does not have downloadable releases.
[INFO] (vz-logger) Data has been updated for this package.
[INFO] (vz-pytorch) Data has been updated for this package.
[INFO] (vzaar) This package is up to date.
[INFO] (vzlog) This package is up to date.
[INFO] (vznncv-cubemx-tools) This package is up to date.
[INFO] (vztcdpchaos-aws) Data has been updated for this package.
[INFO] (vztcdpchaos-htmlreport) Data has been updated for this package.
[INFO] (vztcdpchaos-network) Data has been updated for this package.
[INFO] (vztcdpchaos-report) Data has been updated for this package.
[INFO] (vztcdpchaos-resource) Data has been updated for this package.
[INFO] (vztcdpchaos-slack) Data has been updated for this package.
[INFO] (vztcdpchaos-splunk) Data has been updated for this package.
[INFO] (vzzual) This package is up to date.
[INFO] 
[INFO] 
[INFO] Total packages: 2714
[INFO]     Packages processed: 2714
[INFO]         Packages updated: 688
[INFO]         Packages up-to-date: 1861
[INFO]         Packages without response: 44
[INFO]         Packages without urls: 0
[INFO]         Packages without downloads: 105
[INFO]         Packages with data errors: 16
[INFO]     Packages not processed: 0
[INFO] 
[INFO] 
[INFO] Ending execution.
